feature/#11: Albidaya wa Nihaya Vol 2

segullshairbutt opened the pull request:
Adds the template of the book

segullshairbutt committed:
feature/#11: Albidaya wa Nihaya Vol 2

Adds the template of the book

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

asdf

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

باب 1، قصہ حز قیل تک پورا کیا ہے۔

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

Chapter 1. Qissa Hizqeel tak complete proof read kr k send submit kr rha hon.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

Qisa shamwail as complete kr diya h.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

قصہ داؤد ؑ ان کے زمانے کے حالات complete kr diya h after proof read.

Sajidbhatti526 committed:
feature/#11:Albidaya wa Nihaya Vol 2

داؤد علیہ السلام کی کمیت حیات و کیفیت وفات
chapter 1. yahan tk sara proof read ky bad posh kr rha hon.
Note: Gozishta sari headings bhi hashes ky or spacing ko reset kr diya h.

Sajidbhatti526 committed:
feature/11feature/11-albidaya-wa-nihaya-vol-2

Qissa Suleman hin dawood as complete proof read kr k push kr diya h.

Sajidbhatti526 committed:
feature/#11: Albiadaya wa Nihaya Vol 2

Chapter no 2. k first 2 points proof read kr k push kr rha hon.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

Chapter 1 main Suleman as ky waqia main page 32-33 miss hain jinhin maine Syed Jamal sir ki dictation k mutabik highlight kr diya h.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

chapter 2. Bait ul Maqdas ki tabahi ka zikar. proofread k bad push kr rha hon.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

chapter 2 k 2 topics push kr rha hon. Zikar e Danyal as or is sy agla.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

Qissa Aziz or Qissa Zikria o Yahya as. dono proofread kr k push kr rha hon.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

Chapter 2 k last two topix PF kr k push kr diye hain.

Sajidbhatti526 committed:
feature/#11: Abidya wa Nihaya Vol 2

chapter No. 2. complete ho gya h. PR kr k push kr diya h.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

chapter No.3. ka 1st topic PR kr k push kr duya h.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

chap 3. ky mazeed 2 topix PR kr k push kr diye hain.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

chap 3. k mazeed topix PR kr k push kr rha hon.

Sajidbhatti526 committed:
feature:/#11: Abidaya wa Nihaya Vol 2

cahp 3. k 3 topix PR kr k push kr diye hain.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

chap 3 complete PR kr k push kr diya h.

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,220 @@
+
+## موسی علی السلام کے بعد انبیائے بنی اسرائیل میں سے ایک جماعت کا ذکر
+
+  موسی ؑکے بعد ہم یعنی ہم مورخین عموما داؤد علائے کی نبوت کا ذکر کرتے ہیں لیکن بنی اسرائیل میں نبوت کی ترتیب کا لحاظ رکھا جائے تو ان سے پہلے یوشع ؑ اور کا لب آتے ہیں۔ یوشع و کالب دونوں یوفنا کے بیٹے اورموسٰیؑ کے اصحاب میں شامل تھے۔ ان دونوں میں یوشع ٗموسیؑ کے بہنوئی یعنی مریمؑ کے شوہر تھے اور ان دونوں میں کا لب اور چند دوسرے لوگوں کے علاوہ وہی واحد شخص تھے جو اللہ تعالی سے خائف رہتے تھے۔
```
1۔ "موسی ؑکے بعد ہم یعنی ہم مورخین عموما داؤد علائے کی نبوت کا ذکر کرتے ہیں..." (یہاں "داؤد علائے" غلط لکھا گیا ہے، "داؤد علیہ السلام" درست ہے. )
2۔ اسی لائن میں "عموما" غلط ہے، درست لفظ "عموماً" ہے
3۔ "لیکن بنی اسرائیل میں نبوت کی ترتیب کا لحاظ رکھا جائے تو ان سے پہلے یوشع ؑ اور کا لب آتے ہیں۔" (یہاں لفظ "کالب"  میں "کا" اور "لب" میں اسپیس remove کریں)
4۔ "یوشع و کالب دونوں یوفنا کے بیٹے اورموسٰیؑ کے اصحاب میں شامل تھے۔" (یہاں "اور" اور "موسٰیؑ" کے درمیان اسپیس add کریں)
5۔ "اور ان دونوں میں کا لب اور چند دوسرے لوگوں کے علاوہ وہی واحد شخص تھے۔۔۔" (یہاں لفظ "کالب"  میں "کا" اور "لب" میں اسپیس remove کریں)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,220 @@
+## قصہ حز قیل
+
+(قرآن شریف میں ) اللہ تعالیٰ نے ارشاد فرمایا:
+            "کیا تم نے نہیں دیکھا ان لوگوں کی طرف جو اپنے شہر سے نکلے تھے ان کی تعداد ہزاروں تھی ( لیکن ) وہ موت                  سے  خائف تھے۔ پس اللہ تعالی نے انہیں حکم دیا کہ مر جاؤ پھر انہیں زندہ کر دیا اللہ تعالی انسانوں پر مہربانی فرمانے والا ہے۔ لیکن کثر لوگ (اس کا نا شکر ادا نہیں کرتے)"۔
```
اللہ تعالی انسانوں پر مہربانی فرمانے والا ہے۔ لیکن کثر لوگ (اس کا نا شکر ادا نہیں کرتے)'۔ (یہاں دو جگہ غلطیاں دیکھی گئی:
1: آپ کو "کثر لوگ" لکھا ہے جو کہ غلط ہے، "اکثر لوگ" درست ہو گا"
2: "نا شکر " غلط لکھا ہے، یہاں جملے کی مناسبت سے لفظ "شکر " آئے گا"

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,220 @@
+
+(قرآن شریف میں ) اللہ تعالیٰ نے ارشاد فرمایا:
+            "کیا تم نے نہیں دیکھا ان لوگوں کی طرف جو اپنے شہر سے نکلے تھے ان کی تعداد ہزاروں تھی ( لیکن ) وہ موت                  سے  خائف تھے۔ پس اللہ تعالی نے انہیں حکم دیا کہ مر جاؤ پھر انہیں زندہ کر دیا اللہ تعالی انسانوں پر مہربانی فرمانے والا ہے۔ لیکن کثر لوگ (اس کا نا شکر ادا نہیں کرتے)"۔
+ محمد بن اتفق وہب بن منبہ کے حوالے سے بیان کرتے ہیں کہ جب یوشع ؑ کے کے بعد کا لب بن یوفنا بھی داعی اجل کو لبیک کہہ چکے تو ان کے بعد بنی اسرائیل میں جو قابل ذکر شخصیت رہ گئی وہ جز قبل ( علیہ السلام) کی تھی اور جیسا کہ پہلے ذکر کیا جا چکا ہے وہ حز قیل( علیہ السلام)  ہی تھے جنہوں نے اللہ تعالیٰ سے دعا کی تھی جس کے نتیجے میں جیسا کہ مندرجہ بالا قرآنی آیت کے حوالے سے ابھی بیان کیا کیا کہ وہ ہزاروں آدم ...
```
1. "محمد بن اتفق وہب بن منبہ کے حوالے سے بیان کرتے ہیں کہ جب یوشع ؑ کے کے بعد کا لب بن یوفنا بھی داعی اجل کو لبیک کہہ چکے..."
(اس لائن میں چند غلطیاں دیکھی گئی ہیں جو یہ ہیں:
1.1۔ (یہاں "محمد بن اتفق" نام بلکل غلط لکھا گیا ہے، کتاب پر یہ نام محمد بن اسحٰق" ہے، برائے کرم اسکی درستگی فرمائیں)
1.2۔ آپ کو لکھا ہے ""یوشع ؑ کے کے بعد"، یہاں "کے" ایک اضافی لکھا گیا ہے اسکو remove کریں،
1.3۔ اسی لائن میں لفظ "کالب" ہے، میں "کا" اور "لب" کے درمیان اسپیس ڈالی گئی ہے۔ اسپیس remove کریں۔

2۔ "تو ان کے بعد بنی اسرائیل میں جو قابل ذکر شخصیت رہ گئی وہ جز قبل ( علیہ السلام) کی تھی.." (یہاں آپ نے لکھا ہے "جز قبل" جو کے غلط لکھا گیا ہے، اسکو "حزقیل (علیہ السلام)" لکھ کر درست کریں۔


3۔ "...اور جیسا کہ پہلے ذکر کیا جا چکا ہے وہ حز قیل( علیہ السلام) ہی تھے جنہوں نے اللہ تعالیٰ سے دعا کی تھی..." (یہاں لفظ "حزقیل" میں "حز" اور "قیل" کے درمیان اسپیس اضافی ڈالی گئی ہے، ریمو کریں)

4۔ "جس کے نتیجے میں جیسا کہ مندرجہ بالا قرآنی آیت کے حوالے سے ابھی بیان کیا کیا کہ وہ ہزاروں آدمی جو موت کے خوف سے اپنے شہر سے نکل بھاگے تھے کیونکہ وہاں وبا پھیل گئی تھی"۔ (یہاں آپ نے لکھا ہے "بیان کیا کیا"، اسکو درست کریں "بیان کیا گیا" لکھیں)

5۔ "لیکن اس کے باوجود قضائے الہی سے مر گئے تھے زندہ ہو گئے تھے۔" (یہاں "مر گئے تھے، زندہ ہو گئے تھے۔" comma add کریں)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,220 @@
+(قرآن شریف میں ) اللہ تعالیٰ نے ارشاد فرمایا:
+            "کیا تم نے نہیں دیکھا ان لوگوں کی طرف جو اپنے شہر سے نکلے تھے ان کی تعداد ہزاروں تھی ( لیکن ) وہ موت                  سے  خائف تھے۔ پس اللہ تعالی نے انہیں حکم دیا کہ مر جاؤ پھر انہیں زندہ کر دیا اللہ تعالی انسانوں پر مہربانی فرمانے والا ہے۔ لیکن کثر لوگ (اس کا نا شکر ادا نہیں کرتے)"۔
+ محمد بن اتفق وہب بن منبہ کے حوالے سے بیان کرتے ہیں کہ جب یوشع ؑ کے کے بعد کا لب بن یوفنا بھی داعی اجل کو لبیک کہہ چکے تو ان کے بعد بنی اسرائیل میں جو قابل ذکر شخصیت رہ گئی وہ جز قبل ( علیہ السلام) کی تھی اور جیسا کہ پہلے ذکر کیا جا چکا ہے وہ حز قیل( علیہ السلام)  ہی تھے جنہوں نے اللہ تعالیٰ سے دعا کی تھی جس کے نتیجے میں جیسا کہ مندرجہ بالا قرآنی آیت کے حوالے سے ابھی بیان کیا کیا کہ وہ ہزاروں آدم ...
+ہوا یہ تھا کہ جب وہ اپنے شہر سے نکل کر صعید پہنچے تھے تو اللہ تعالیٰ کے حکم سے وہ سب کے سب یک لخت مر گئے تھے اور زمین میں ان کا گوشت ان کی ہڈیوں سے جدا ہو کر خاک میں مل چکا تھا تا ہم لوگوں نے اس سے قبل ان سب کو ایک جگہ دفن کر دیا تھا تا کہ درندے ان کا گوشت نہ کھا جائیں اور پھر اس جگہ ایک خبر ہ ( مقبرہ) بھی بنا دیا گیا تھا اور اس واقعے کو مد میں گزرگئی تھیں لیکن جب من قبل علائے کو اللہ تعالٰی نے ن ...
```
1۔ "اور زمین میں ان کا گوشت ان کی ہڈیوں سے جدا ہو کر خاک میں مل چکا تھا تا ہم لوگوں نے اس سے قبل ان سب کو ایک جگہ دفن کر دیا تھا" (یہاں "تا ہم" غلط ہے، "تاہم" درست ہے) اسپیس نہیں آتی درمیان میں

2۔ "اور پھر اس جگہ ایک خبر ہ ( مقبرہ) بھی بنا دیا گیا تھا" (یہاں "خبرہ " غلط لکھا گیا ہے، کتاب میں "خطبرہ" ہے، اسکو صحیح لکھیں)

3۔ "اور اس واقعے کو مد میں گزرگئی تھیں لیکن جب من قبل علائے کو اللہ تعالٰی نے نبوت سے سرفراز فرمایا." ( یہاں "مد میں" غلط لکھا گیا ہے، درست لفظ "مدتیں" ہے)۔ اسی لائن میں آپ نے لکھا ہے "من قبل علائے"  جو کے بلکل غلط ہے، درست لفظ ہے "حزقیل علیہ السلام" ہے۔

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,220 @@
+            "کیا تم نے نہیں دیکھا ان لوگوں کی طرف جو اپنے شہر سے نکلے تھے ان کی تعداد ہزاروں تھی ( لیکن ) وہ موت                  سے  خائف تھے۔ پس اللہ تعالی نے انہیں حکم دیا کہ مر جاؤ پھر انہیں زندہ کر دیا اللہ تعالی انسانوں پر مہربانی فرمانے والا ہے۔ لیکن کثر لوگ (اس کا نا شکر ادا نہیں کرتے)"۔
+ محمد بن اتفق وہب بن منبہ کے حوالے سے بیان کرتے ہیں کہ جب یوشع ؑ کے کے بعد کا لب بن یوفنا بھی داعی اجل کو لبیک کہہ چکے تو ان کے بعد بنی اسرائیل میں جو قابل ذکر شخصیت رہ گئی وہ جز قبل ( علیہ السلام) کی تھی اور جیسا کہ پہلے ذکر کیا جا چکا ہے وہ حز قیل( علیہ السلام)  ہی تھے جنہوں نے اللہ تعالیٰ سے دعا کی تھی جس کے نتیجے میں جیسا کہ مندرجہ بالا قرآنی آیت کے حوالے سے ابھی بیان کیا کیا کہ وہ ہزاروں آدم ...
+ہوا یہ تھا کہ جب وہ اپنے شہر سے نکل کر صعید پہنچے تھے تو اللہ تعالیٰ کے حکم سے وہ سب کے سب یک لخت مر گئے تھے اور زمین میں ان کا گوشت ان کی ہڈیوں سے جدا ہو کر خاک میں مل چکا تھا تا ہم لوگوں نے اس سے قبل ان سب کو ایک جگہ دفن کر دیا تھا تا کہ درندے ان کا گوشت نہ کھا جائیں اور پھر اس جگہ ایک خبر ہ ( مقبرہ) بھی بنا دیا گیا تھا اور اس واقعے کو مد میں گزرگئی تھیں لیکن جب من قبل علائے کو اللہ تعالٰی نے ن ...
+اسباط نے بھی اسدی ابی مالک ابی صالح، ابن عباس 'مرہ ابن مسعود اور کچھ صحابہ رضی اللہ عنہ کے حوالے سے مندرجہ بالا قرآنی آیہ شریفہ کا مفہوم بیان کرتے ہوئے اس واقعے کا ذکر کیا ہے اور بتایا ہے کہ ایک گاؤں جس کا نام داور دان لیا جاتا تھا اور وہ واسط سے قبل آباد تھا میں طاعون پھیل گیا تو بہت سے لوگ وہاں سے بھاگ کر ایک نزدیکی علاقے میں چلے گئے تھے ۔ تاہم جو لوگ و بار سے بھاگنا مناسب نہ سمجھ کر اور مشیت ا ...
```
1۔ "اور بتایا ہے کہ ایک گاؤں جس کا نام داور دان لیا جاتا تھا اور وہ واسط سے قبل آباد تھا میں طاعون پھیل گیا." (یہاں گاؤں کے نام "داور دان" میں اسپیس نہیں آتی، اسکو "داوردان" لکھیں)

2۔ "تاہم جو لوگ و بار سے بھاگنا مناسب نہ سمجھ کر اور مشیت ایزدی کو آمنا و صدقنا کہہ کر وہیں رُکے رہے تھے ان میں سے اکثر اس وبا کا شکار ہونے سے بچ گئے تھے." (یہاں شروع میں لکھا ہے کہ "کو لوگ و بار " ، یہاں و بار غلط لکھا ہے، اس کی جگہ "وہاں" آتا ہے,
درست کریں)

Laaibah replied:
1۔ "اور بتایا ہے کہ ایک گاؤں جس کا نام داور دان لیا جاتا تھا اور وہ واسط سے قبل آباد تھا میں طاعون پھیل گیا." (یہاں گاؤں کے نام "داور دان" میں اسپیس نہیں آتی، اسکو "داوردان" لکھیں)

2۔ "تاہم جو لوگ و بار سے بھاگنا مناسب نہ سمجھ کر اور مشیت ایزدی کو آمنا و صدقنا کہہ کر وہیں رُکے رہے تھے ان میں سے اکثر اس وبا کا شکار ہونے سے بچ گئے تھے." (یہاں شروع میں لکھا ہے کہ "کو لوگ و بار " ، یہاں و بار غلط لکھا ہے، اس کی جگہ "وہاں" آتا ہے,
درست کریں)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,220 @@
+ محمد بن اتفق وہب بن منبہ کے حوالے سے بیان کرتے ہیں کہ جب یوشع ؑ کے کے بعد کا لب بن یوفنا بھی داعی اجل کو لبیک کہہ چکے تو ان کے بعد بنی اسرائیل میں جو قابل ذکر شخصیت رہ گئی وہ جز قبل ( علیہ السلام) کی تھی اور جیسا کہ پہلے ذکر کیا جا چکا ہے وہ حز قیل( علیہ السلام)  ہی تھے جنہوں نے اللہ تعالیٰ سے دعا کی تھی جس کے نتیجے میں جیسا کہ مندرجہ بالا قرآنی آیت کے حوالے سے ابھی بیان کیا کیا کہ وہ ہزاروں آدم ...
+ہوا یہ تھا کہ جب وہ اپنے شہر سے نکل کر صعید پہنچے تھے تو اللہ تعالیٰ کے حکم سے وہ سب کے سب یک لخت مر گئے تھے اور زمین میں ان کا گوشت ان کی ہڈیوں سے جدا ہو کر خاک میں مل چکا تھا تا ہم لوگوں نے اس سے قبل ان سب کو ایک جگہ دفن کر دیا تھا تا کہ درندے ان کا گوشت نہ کھا جائیں اور پھر اس جگہ ایک خبر ہ ( مقبرہ) بھی بنا دیا گیا تھا اور اس واقعے کو مد میں گزرگئی تھیں لیکن جب من قبل علائے کو اللہ تعالٰی نے ن ...
+اسباط نے بھی اسدی ابی مالک ابی صالح، ابن عباس 'مرہ ابن مسعود اور کچھ صحابہ رضی اللہ عنہ کے حوالے سے مندرجہ بالا قرآنی آیہ شریفہ کا مفہوم بیان کرتے ہوئے اس واقعے کا ذکر کیا ہے اور بتایا ہے کہ ایک گاؤں جس کا نام داور دان لیا جاتا تھا اور وہ واسط سے قبل آباد تھا میں طاعون پھیل گیا تو بہت سے لوگ وہاں سے بھاگ کر ایک نزدیکی علاقے میں چلے گئے تھے ۔ تاہم جو لوگ و بار سے بھاگنا مناسب نہ سمجھ کر اور مشیت ا ...
+اس واقع کا ذکر کرتے ہوئے باسط بیان کرتے ہیں کہ داور دان کے لوگ دوسری بار جہاں بھاگ کر گئے تھے وہ جگہ قابل کہلاتی تھی اور وہاں پہنچنے والوں کی تعداد ہزاروں پر مشتمل تھی کچھ لوگوں نے ان کی تعداد تمیں ہزار بتائی ہے۔ قابل ایک وسیع و عریض وادی تھی جس کے نشیبی حصے سے ایک غیبی آواز آئی تھی کہ "مرجاؤ "اور وہ سب مر گئے تھے ۔ البتہ جب حز قلس( علیہ السلام) کو مدتوں بعد وہاں سے گزر ہوا تھا اور وہ وہاں کے لوگ ...
```
1۔ "کچھ لوگوں نے ان کی تعداد تمیں ہزار بتائی ہے۔" (یہاں "تمیں ہزار" غلط لکھا ہے، درست لفظ "تیس ہزار" ہے)

2۔ "قابل ایک وسیع و عریض وادی تھی جس کے نشیبی حصے سے ایک غیبی آواز آئی تھی کہ 'مرجاؤ' اور وہ سب مر گئے تھے۔" (یہاں "مر جاؤ" میں "مر" اور "جاؤ" کے درمیان میں اسپیس add کریں)

3۔ "البتہ جب حز قلس( علیہ السلام) کو مدتوں بعد وہاں سے گزر ہوا تھا." ( یہاں بھی آپ نے یہ لفظ غلط لکھا ہے، آپ نے لکھا ہے "حز قلس" جب کہ درست لفظ ہے "حزقیل", درست کریں)

4۔ "اللہ تعالیٰ نے ان مردوں کی واحد غیر سے انہیں دوبارہ زندگی عطا فر ما کردہ قبیلن کے سامنے کھڑا کر دیا تھا." (یہاں "واحد غیر" لکھا ہے آپ نے، یہاں "واحد قبر" آتا ہے، کتاب پر miss print ہوا ہے)۔ اسی لائن میں ایک اور غلطی بھی دیکھی گئی ہے، آپ نے لکھا ہے "زندگی عطا فر ما کردہ قبیلن" یہ بلکل غلط لکھا گیا ہے، درست phrase یہ ہے:  "زندگی عطا فرما کر حزقیل" یہاں بھی درستگی درکار ہے.

5۔ "او رجہ یا کہ محمد بن اسحٰق کی بیان کرد وایت میں پہلے تایا جا چکا ہے." ( یہ آپ نے شروع سے ہی غلط لکھی ہے، میں درست جملہ لکھا ہے، برائے کرم غور فرمائیں: "اور جیسا کہ محمد بن اسحٰق کی بیان کردہ روایت میں پہلے بتایا جا چکا ہے."

6۔  "اور پھر انہوں نے یک زبان ہو کر سبحانک اللهُم وَ بِحَمْدِكَ لَا إِلَهَ إِلَّا أَنتَ(ترجمہ:اے خدا تو پاک ہے اور ترری حمد کے ساتھ ترهے سوا کوئی معبود نہںی۔)کیا تھا۔" (یہاں آپ کو ترجمہ لکھنے میں بہت غلطیاں کی ہیں, درست کریں »» اس کا ترجمہ ہے: "پاک ہے تو اے اللہ! اور تیری حمد کے ساتھ، تیرے سوا کوئی معبود نہیں۔"

7۔ "حالانکہ اس وقت تک ان کا گوشت تو گوشت ان کی پڑیاں تک سرمہ ہو کر خاک میں مل چکی تھیں لیکن اب وہ زندہ ہو کر اپنی قوم میں چلے گئے تھے." (یہاں "پڑیاں" لکھا ہوا ہے، جو کہ غلط ہے، درست لفظ "ہڈیاں" ہے)

Laaibah reviewed (changes requested):
I have started reviewing your work for Volume 2, and I am quite disappointed with the quality. I have only read the first 16 lines so far, and I have already found a significant number of mistakes. I request you to carefully review your entire work yourself, correct all errors, and then update me accordingly. Please ensure accuracy before submitting further work.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

Chap #4. k four topix PR kr k push kiye hain.

segullshairbutt committed:
Merge branch 'main' into feature/11-albidaya-wa-nihaya-vol-2

segullshairbutt committed:
feature/#11-albidaya-wa-nihaya-vol-2

Adds the functionality of deploying previews.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

chap #1 complete RV kr k push kr diya h.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

chap #2 complete re-PR kr k push kr diya h.

Sajidbhatti526 committed:
feature/11#: Albidaya wa Nihaya Vol 2

chap #3 & 4. re-PR kr k push kr diya h.

Sajidbhatti526 committed:
feature/11#: Albidaya wa Niahaya Vol 2

Chap # 2. main bhai thori or correction kr k push kr diya h.

Sajidbhatti526 committed:
feature/#11: Abidaya wa Nihaya Vol 2

chap # 2,3 $ 4 main choti c mazeem correciton ki h.

Sajidbhatti526 committed:
feature/#11: Abidaya wa Nihaya Vol 2

chap #4 ky mazeed 7 topix deep PR kr k push kr diye hain. or chap#2 main kuch correction ki h.

Sajidbhatti526 committed:
feature/#11: Abidaya wa Nihaya Vol 2

chap #4&5 complete PR kr k or chap #6 k 2 topix PR kr k push kr diye hn

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

chap #5 main peragraph or baab ki numbering urdu wali change ki h. kindly check plz.....

Sajidbhatti526 committed:
feature/#11: Abidaya wa Nihaya Vol 2

All chaps numbering change into Urdu numbering

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+یہ بھی کہا جاتا ہے کہ دور طالوت سے قبل مذکورہ بالا تابوت بنی اسرائیل کے قبضے میں تھا جو اصنام پرست تھے لیکن انہوں نے دیکھا کہ وہ تابوت خود بخود فرش سے اٹھ کر ان کے اس بت بعل کے سر پر پہنچ جاتا ہے جس کی وہ پرستش کرتے تھے۔ وہ اسے وہاں سے اتار کر فرش پر رکھتے لیکن اگلی صبح وہ پھر وہیں پہنچ جاتا۔ اس لیے انہوں نے تنگ آکر اسے دو آدمیوں کے ہاتھ بنی اسرائیل میں بھجوا دیا تھا جسے بنی اسرائیل اپنے نبی کے ا ...
+ابن عباس رضی اللہ عنہ کے علاوہ متعدد مفسرین نے بھی بیان کیا ہے کہ شمویل (علیہ السلام) کے قصے میں جس نہر  کا ذکر آیا ہے اور جس کا پانی پینے سے تھوڑی سی مقدار کے علاوہ بنی اسرائیل کو منع کیا گیا تھا اس کا نام کلام الہٰی کے مطابق نہر الاردن تھا اور وہ آج تک اس نام سے یعنی نہر اردن یا دریائے اردن کے نام سے مشہور چلا آتا ہے۔
+السدی کہتے ہیں کہ بنی اسرائیل کے لشکر کی تعداد اسی ہزار جوانوں اور امرائے لشکر پر مشتمل تھی جن پر چھہتر ہزار نے حکم الہٰی کے خلاف دریائے اردن کا پانی پی لیا تھا اور طالوت کے ہمراہ صرف چار ہزار افراد رہ گئے تھے جب کہ بخاریؒ نے صحیح بخاری میں قصہ بنی اسرائیل کے ضمن میں اور زہیر و ثوری نے ابی اسحٰق' البراء بن عازب کے حوالے سے بتایا ہے کہ طالوت کے ہمراہ اس کے جن فوجیوں نے دریائے اردن کو عبور کیا تھا  ...
+ طالوت کے قلیل التعداد لشکریوں نے بھی جو اہل ایمان تھے اصحاب بدر کی طرح اللہ تعالیٰ سے وہی دعا کی تھی کہ(یا اللہ ہم پرصبر کے دہانے کھول دے' ہمیں ثابت قدم رکھ اور کفار پر فتح کے لیے ہماری امداد فرما) اور چونکہ اس دعا کے وقت ان کا ظاہر و باطن یکساں تھا اور وہ خدا کی نصرت پر اعتماد ر کھتے تھے اس لیے اللہ تعالیٰ نے ان کے حسبِ مراد ان کے کثیر التعداد دشمن پر انہیں فتح بخشی جیسا کہ قرآن میں ارشاد ہوا ک ...
```
اور چونکہ اس دعا کے وقت ان کا ظاہر و باطن یکساں تھا اور وہ خدا کی نصرت پر اعتماد ر کھتے تھے اس لیے اللہ تعالیٰ نے ان کے حسبِ مراد ان کے کثیر التعداد دشمن پر انہیں فتح بخشی

غلطی: ر کھتے
درست: رکھتے (اضافی وقف )

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+ابن عباس رضی اللہ عنہ کے علاوہ متعدد مفسرین نے بھی بیان کیا ہے کہ شمویل (علیہ السلام) کے قصے میں جس نہر  کا ذکر آیا ہے اور جس کا پانی پینے سے تھوڑی سی مقدار کے علاوہ بنی اسرائیل کو منع کیا گیا تھا اس کا نام کلام الہٰی کے مطابق نہر الاردن تھا اور وہ آج تک اس نام سے یعنی نہر اردن یا دریائے اردن کے نام سے مشہور چلا آتا ہے۔
+السدی کہتے ہیں کہ بنی اسرائیل کے لشکر کی تعداد اسی ہزار جوانوں اور امرائے لشکر پر مشتمل تھی جن پر چھہتر ہزار نے حکم الہٰی کے خلاف دریائے اردن کا پانی پی لیا تھا اور طالوت کے ہمراہ صرف چار ہزار افراد رہ گئے تھے جب کہ بخاریؒ نے صحیح بخاری میں قصہ بنی اسرائیل کے ضمن میں اور زہیر و ثوری نے ابی اسحٰق' البراء بن عازب کے حوالے سے بتایا ہے کہ طالوت کے ہمراہ اس کے جن فوجیوں نے دریائے اردن کو عبور کیا تھا  ...
+ طالوت کے قلیل التعداد لشکریوں نے بھی جو اہل ایمان تھے اصحاب بدر کی طرح اللہ تعالیٰ سے وہی دعا کی تھی کہ(یا اللہ ہم پرصبر کے دہانے کھول دے' ہمیں ثابت قدم رکھ اور کفار پر فتح کے لیے ہماری امداد فرما) اور چونکہ اس دعا کے وقت ان کا ظاہر و باطن یکساں تھا اور وہ خدا کی نصرت پر اعتماد ر کھتے تھے اس لیے اللہ تعالیٰ نے ان کے حسبِ مراد ان کے کثیر التعداد دشمن پر انہیں فتح بخشی جیسا کہ قرآن میں ارشاد ہوا ک ...
+”اللہ نے بدر میں تمہاری مدد کی جب کہ تم (قلیل تعداد میں) کمزور تھے اس لیے اللہ سے ڈرتے رہو تا کہ تم شکر گزاربن سکو“۔
```
اللہ نے بدر میں تمہاری مدد کی جب کہ تم (قلیل تعداد میں) کمزور تھے اس لیے اللہ سے ڈرتے رہو تا کہ تم شکر گزاربن سکو

غلطی: شکر گزاربن
درست: شکر گزار بن (درمیان میں اسپیس ضروری ہے)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+”اللہ نے بدر میں تمہاری مدد کی جب کہ تم (قلیل تعداد میں) کمزور تھے اس لیے اللہ سے ڈرتے رہو تا کہ تم شکر گزاربن سکو“۔
+ پھر جیسا کہ اللہ تعالیٰ کا ارشاد ہے طالوت کے تھوڑے سے ساتھیوں کے ہاتھوں جالوت قتل ہوا اور اللہ تعالیٰ نے طالوت کو نہ صرف بادشاہت بخشی بلکہ اسے علم و حکمت کی دولت سے بھی مالا مال کر دیا۔
+اس قصے میں داؤد (علیہ السلام) کی شجاعت کا بھی مدلل ثبوت ملتا ہے جنہوں نے جالوت کو قتل کیا تھا اور چونکہ ان کی حریف فوج کو نہ صرف شکست ہوئی تھی بلکہ دشمنوں کا بادشاہ بھی چونکہ میدان جنگ میں مارا گیا تھا اس لیے سامانِ حرب و ضرب کے علاوہ اس کا دیگر سامان اور زر و جواہر بھی کثیر تعداد میں ان کے ہاتھ آئے تھے۔ اس سے یہ بھی ثبوت ملتا ہے کہ حق اور حق پرست نصرت الہٰی سے کس طرح باطل اور باطل پرستوں پر غالب ...
+السدی کی روایت میں یہ ذکر بھی آیا ہے کہ داؤد علیہ السلام  اپنے والد کے تیرا بیٹوں میں سب سے چھوٹے تھے اور ان میں سے دو یہ بیان کر چکے تھے کہ انہوں نے سنا ہے کہ طالوت نے اعلان کیا ہے کہ بنی اسرائیل میں سے جو شخص جالوت کو لڑائی میں قتل کرے گا وہ اس کے ساتھ اپنی بیٹی کی شادی کرنے کے علاوہ اسے اپنی سلطنت میں نصف کا شریک بھی کرے گا یعنی اس طرح وہ بنی اسرائیل کی حوصلہ افزائی کیا کرتا تھا۔ اس کے علاوہ د ...
```
غلطی: "تیرا بیٹوں"
درست: "تیرہ بیٹوں" (لفظ "تیرہ" کا درست املا)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+السدی کی روایت میں یہ ذکر بھی آیا ہے کہ داؤد علیہ السلام  اپنے والد کے تیرا بیٹوں میں سب سے چھوٹے تھے اور ان میں سے دو یہ بیان کر چکے تھے کہ انہوں نے سنا ہے کہ طالوت نے اعلان کیا ہے کہ بنی اسرائیل میں سے جو شخص جالوت کو لڑائی میں قتل کرے گا وہ اس کے ساتھ اپنی بیٹی کی شادی کرنے کے علاوہ اسے اپنی سلطنت میں نصف کا شریک بھی کرے گا یعنی اس طرح وہ بنی اسرائیل کی حوصلہ افزائی کیا کرتا تھا۔ اس کے علاوہ د ...
+”مجھے اٹھا لو اور جالوت پر پھینکو تو مجھی سے تم جالوت کو قتل کر دو گے“۔
+کچھ مؤرخین نے یہ بھی لکھا ہے کہ جالوت سے طالوت کی جنگ کے موقع پر داؤدؑ کو پتھر سے جو آواز سنائی دی تھی وہ در حقیقت ایک کے بجائے تعداد میں تین تھے اور داؤدؑ نے انہیں اٹھا کر اپنے گوپھن میں رکھ لیا تھا تو وہ تینوں ایک پتھر بن گئے تھے۔ پھر یہ ہوا کہ مذکورہ بالا لڑائی میں جب جالوت داؤدؑ کے سامنے آیا تو انہوں نے اس سے فرمایا کہ ”میرے سامنے سے ہٹ جا' میں تجھے قتل کرنا نہیں چاہتا“ داؤدؑ سے یہ سن کر جالو ...
+کہتے ہیں کہ جالوت پر فتح یاب ہونے کے بعد طالوت نے حسب اعلان واؤدؑ سے اپنی بیٹی بیاہ دی اور انہیں حسب وعد ہ اپنی سلطنت میں بھی نصف کا شریک کر لیا لیکن داؤدؑ کی اس شجاعت و دلیری کی وجہ سے بنی اسرائیل طالوت سے کہیں زیادہ ان کی عزت کرنے لگے جسے دیکھ کر طالوت رشک و حسد کا شکار ہو گیا اور اس نے داؤدؑ کو قتل کرنا چاہا تو بنی اسرائیل کے علماء نے اسے اس سے روکا لیکن طالوت نے ان علماء میں سے اکثر کوقتل کرا ...
```
غلطی: "وعد ہ"
درست: "وعدہ" (درمیان میں غیر ضروری فاصلہ ہے، درست املا "وعدہ" ہے)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+”اے طالوت تو نے ہمیں قتل کرایا تھا اور بظاہر اب ہم مردہ ہیں لیکن در حقیقت ہم زندہ ہیں“۔
+زمین سے یہ آواز سن کر طالوت اور زیادہ خوفزدہ ہو گیا اور پہلے سے زیادہ گریہ و زاری کرنے لگا لیکن ایک روز جب اس نے اپنے کسی قریبی ساتھی کو اس حقیقت سے آگاہ کیا تو اس نے پوچھا:
+”ان علماء میں سے جنہیں آپ نے قتل کرایا تھا کوئی باقی ہے یا نہیں؟“۔
+اس سوال کے جواب میں اس نے بنی اسرائیل کے باقی ماندہ علماء کی تلاش شروع کی تو آخر کا را سے ایسے عالم کا پتہ معلوم ہوا جو اس وقت گوشہ نشینی کی زندگی بسر کر رہا تھا۔
```
غلطی: "آخر کا را سے"
درست: "آخر کار اسے" (اصل لفظ "آخر کار" ہے، "کا را" غلط املا ہے)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+زمین سے یہ آواز سن کر طالوت اور زیادہ خوفزدہ ہو گیا اور پہلے سے زیادہ گریہ و زاری کرنے لگا لیکن ایک روز جب اس نے اپنے کسی قریبی ساتھی کو اس حقیقت سے آگاہ کیا تو اس نے پوچھا:
+”ان علماء میں سے جنہیں آپ نے قتل کرایا تھا کوئی باقی ہے یا نہیں؟“۔
+اس سوال کے جواب میں اس نے بنی اسرائیل کے باقی ماندہ علماء کی تلاش شروع کی تو آخر کا را سے ایسے عالم کا پتہ معلوم ہوا جو اس وقت گوشہ نشینی کی زندگی بسر کر رہا تھا۔
+طالوت نے اسے بلا کر پہلے تو اپنی سابقہ حرکت پر ندامت کا اظہار کیا اور پھر اس سے پوچھا کہ آیا اس کے اس پچھلے گناہ کی تو بہ کا اللہ تعالیٰ کی بارگاہ میں قبول ہونے کا کیا طریقہ ہے؟
```
غلطی: "تو بہ"
درست: "توبہ" (صحیح املا "توبہ" ہے، اسے الگ لکھنا غلط ہے)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+”ان علماء میں سے جنہیں آپ نے قتل کرایا تھا کوئی باقی ہے یا نہیں؟“۔
+اس سوال کے جواب میں اس نے بنی اسرائیل کے باقی ماندہ علماء کی تلاش شروع کی تو آخر کا را سے ایسے عالم کا پتہ معلوم ہوا جو اس وقت گوشہ نشینی کی زندگی بسر کر رہا تھا۔
+طالوت نے اسے بلا کر پہلے تو اپنی سابقہ حرکت پر ندامت کا اظہار کیا اور پھر اس سے پوچھا کہ آیا اس کے اس پچھلے گناہ کی تو بہ کا اللہ تعالیٰ کی بارگاہ میں قبول ہونے کا کیا طریقہ ہے؟
+اس عالم نے طالوت پر ترس کھا کر ایک ایسی عبادت گزار ضعیف عورت کی نشاندہی کی جو مستجاب الدعوات ہونے میں مشہورتھی۔
```
غلطی: "مشہورتھی"
درست: "مشہور تھی" (درمیان میں وقفہ دینا ضروری ہے)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+”وہ طریقہ یہ ہے کہ یہ (طالوت) یہ ملک چھوڑ دے اور خدا کی راہ میں اس وقت تک کافروں سے جہاد کرتا رہے جب تک قتل نہ ہو جائے اور یہ میت کی شکل میں یہاں واپس نہ آئے“۔ 
+یوشع علیہ السلام کی زبان سے یہ سن کر طالوت نے ان کے سامنے سر اطاعت خم کیا اور اپنا ملک چھوڑ کر سلطنت داؤدؑ کے حوالے کر کے چلا گیا۔ اس کی اولاد میں سے اس وقت صرف تیرہ آدمی تھے جو سب کے سب اس کے ساتھ ہی وہاں سے چلے گئے اور اس وقت تک راہِ خدا میں جہاد کرتے رہے جب تک قتل نہ ہو گئے۔ ان میں خود طالوت بھی شامل تھا۔
+ابن جریر نے اپنی تاریخ میں یہ قصہ السدی کی طرح بہ اسناد روایت کیا ہے لیکن بعض جگہ یہ روایت محل نظر ہے اور قابل قبول نہیں ہو سکتی۔ واللہ اعلم 
+محمد بن اسحٰق فرماتے ہیں کہ بنی اسرائیل کے جس نبی نے طالوت کو بار گاہ باری تعالیٰ میں تو بہ کی قبولیت اور اس کے سابقہ گناہوں کے سلسلے میں تلافی مافات کا طریقہ بتایا تھا وہ یسع بن اخطوب تھے اور وہی طالوت کو یوشعؑ کی قبر پر لے گئے تھے۔ ابن جرید نے بھی اس کا ذکر کیا ہے اور یہی انسب معلوم ہوتا ہے۔ جب کہ ثعلبی کا بیان ہے کہ وہ عورت طالوت کو اشمویلؑ کی قبر پر لے گئی تھی اور وہ اس کی دعا سے زندہ ہو کر ا ...
```
1۔ غلطی: "بار گاہ"
درست: "بارگاہ"

2۔ غلطی: "تو بہ"
درست: "توبہ"

3۔ غلطی: "ابن جرید"
درست: "ابن جریر"

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+یوشع علیہ السلام کی زبان سے یہ سن کر طالوت نے ان کے سامنے سر اطاعت خم کیا اور اپنا ملک چھوڑ کر سلطنت داؤدؑ کے حوالے کر کے چلا گیا۔ اس کی اولاد میں سے اس وقت صرف تیرہ آدمی تھے جو سب کے سب اس کے ساتھ ہی وہاں سے چلے گئے اور اس وقت تک راہِ خدا میں جہاد کرتے رہے جب تک قتل نہ ہو گئے۔ ان میں خود طالوت بھی شامل تھا۔
+ابن جریر نے اپنی تاریخ میں یہ قصہ السدی کی طرح بہ اسناد روایت کیا ہے لیکن بعض جگہ یہ روایت محل نظر ہے اور قابل قبول نہیں ہو سکتی۔ واللہ اعلم 
+محمد بن اسحٰق فرماتے ہیں کہ بنی اسرائیل کے جس نبی نے طالوت کو بار گاہ باری تعالیٰ میں تو بہ کی قبولیت اور اس کے سابقہ گناہوں کے سلسلے میں تلافی مافات کا طریقہ بتایا تھا وہ یسع بن اخطوب تھے اور وہی طالوت کو یوشعؑ کی قبر پر لے گئے تھے۔ ابن جرید نے بھی اس کا ذکر کیا ہے اور یہی انسب معلوم ہوتا ہے۔ جب کہ ثعلبی کا بیان ہے کہ وہ عورت طالوت کو اشمویلؑ کی قبر پر لے گئی تھی اور وہ اس کی دعا سے زندہ ہو کر ا ...
+یہ کسی نبی کا معجزہ تو ہو ناممکن ہے لیکن کسی عورت کا نبیہ ہونا بہر حال ناممکن ہے اور اس سے کسی ایسے معجزے کا منسوب کرنا بعید از قیاس ہے۔ واللہ اعلم
```
"بہر حال"
درست: "بہرحال"

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+ابن جریر نے اپنی تاریخ میں یہ قصہ السدی کی طرح بہ اسناد روایت کیا ہے لیکن بعض جگہ یہ روایت محل نظر ہے اور قابل قبول نہیں ہو سکتی۔ واللہ اعلم 
+محمد بن اسحٰق فرماتے ہیں کہ بنی اسرائیل کے جس نبی نے طالوت کو بار گاہ باری تعالیٰ میں تو بہ کی قبولیت اور اس کے سابقہ گناہوں کے سلسلے میں تلافی مافات کا طریقہ بتایا تھا وہ یسع بن اخطوب تھے اور وہی طالوت کو یوشعؑ کی قبر پر لے گئے تھے۔ ابن جرید نے بھی اس کا ذکر کیا ہے اور یہی انسب معلوم ہوتا ہے۔ جب کہ ثعلبی کا بیان ہے کہ وہ عورت طالوت کو اشمویلؑ کی قبر پر لے گئی تھی اور وہ اس کی دعا سے زندہ ہو کر ا ...
+یہ کسی نبی کا معجزہ تو ہو ناممکن ہے لیکن کسی عورت کا نبیہ ہونا بہر حال ناممکن ہے اور اس سے کسی ایسے معجزے کا منسوب کرنا بعید از قیاس ہے۔ واللہ اعلم
+اہل تو ریت کے نزدیک طالوت کے قتل ہونے تک اس کی مدت حکومت چالیس سال رہی۔
```
غلطی: "اہل تو ریت"
درست: "اہلِ توریت" (اسپیس اضافی ہے)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+
+## قصہ داؤد ؑ ان کے زمانے کے حالات' ان کے فضائل و شمائل' ان کی نبوت کے دلائل اور ذکر اظہار
+
+عربی تواریخ کے مطابق حضرت داؤدؑ کا پورا نام داؤ دبن ایشا بن عوید[1^] بن عابر بن سلمون بن نحشون بن عوينا ذب بن ارم بن حصرون بن فارس ابن یہوذ ا بن یعقوب بن اسحٰق بن ابراہیم خلیل اللہؑ بتایا جاتا ہے جو ارض ِبیت المقدس میں اللہ تعالیٰ کے نبی کی حیثیت سے قیام پذیر رہے۔
```
1۔ غلطی: "داؤ دبن"
درست: "داؤد بن" (یہاں "داؤد" اور "بن" کے درمیان غلطی سے اضافی اسپیس ہے)

2۔ غلطی: "یہوذ ا"
درست: "یہوذا (اسپیس اضافی ہے)

3۔ غلطی: "ارض ِبیت المقدس"
درست: "ارضِ بیت المقدس"

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+
+عربی تواریخ کے مطابق حضرت داؤدؑ کا پورا نام داؤ دبن ایشا بن عوید[1^] بن عابر بن سلمون بن نحشون بن عوينا ذب بن ارم بن حصرون بن فارس ابن یہوذ ا بن یعقوب بن اسحٰق بن ابراہیم خلیل اللہؑ بتایا جاتا ہے جو ارض ِبیت المقدس میں اللہ تعالیٰ کے نبی کی حیثیت سے قیام پذیر رہے۔
+ محمد بن اسحٰق بعض اہلِ علم اور وہب بن منبہ کے حوالے سے بیان کرتے ہیں کہ داؤدؑ کا قد چھوٹا'  ان کی آنکھیں نیلگوں' بال کم لیکن ان کا دل صاف اور طیب و طاہر تھا۔ ان کے ہاتھوں جالوت کے قتل کا ذکر پہلے کیا جا چکا ہے جس کے بارے میں ابن عساکر مزید کہتے ہیں کہ انہوں نے جالوت کو مرج الصفر کے نواح میں قصرام حکیم کے قریب قتل کیا تھا اور اس کے بعد بنی اسرائیل نےانہیں بطور عزت افزائی اپنا بادشاہ بنا لیا اور و ...
+یہ با دشاہت اور نبوت داؤدؑ اور ان کے بیٹے (سلیمانؑ) میں آخرتک قائم رہی یعنی  اللہ تعالیٰ نے انہیں دنیا د آخرت دونوں میں اس یکجا عظمت وفضیلت سے سرفراز فرمایا۔
```
1۔ غلطی: "یہ با دشاہت"
درست: "یہ بادشاہت" (درمیان میں غیر ضروری اسپیس)

2۔ غلطی: "دنیا د آخرت"
درست: "دنیا و آخرت" ("د" کی جگہ "و" درست ہے)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+ محمد بن اسحٰق بعض اہلِ علم اور وہب بن منبہ کے حوالے سے بیان کرتے ہیں کہ داؤدؑ کا قد چھوٹا'  ان کی آنکھیں نیلگوں' بال کم لیکن ان کا دل صاف اور طیب و طاہر تھا۔ ان کے ہاتھوں جالوت کے قتل کا ذکر پہلے کیا جا چکا ہے جس کے بارے میں ابن عساکر مزید کہتے ہیں کہ انہوں نے جالوت کو مرج الصفر کے نواح میں قصرام حکیم کے قریب قتل کیا تھا اور اس کے بعد بنی اسرائیل نےانہیں بطور عزت افزائی اپنا بادشاہ بنا لیا اور و ...
+یہ با دشاہت اور نبوت داؤدؑ اور ان کے بیٹے (سلیمانؑ) میں آخرتک قائم رہی یعنی  اللہ تعالیٰ نے انہیں دنیا د آخرت دونوں میں اس یکجا عظمت وفضیلت سے سرفراز فرمایا۔
+جیسا کہ اس سے قبل جو آیات قرآنی پیش کی گئیں ان میں اللہ تعالیٰ نے داؤدؑ کے ہاتھوں جالوت کے قتل کا ذکر فرماتے ہوئے ارشاد فرمایا کہ روئے زمین پر کسی کو سلطانی بخشنے کا مقصد یہ ہے کہ طاقت ور لوگ کمزوروں پر غلبہ حاصل نہ کرنے پائیں کیونکہ اگر ایسا ہو تو دنیا میں فساد برپا ہو جائے اور طاقت ور لوگ کمزوروں کو ہمیشہ مغلوب رکھیں لیکن اللہ تعالیٰ چونکہ اپنے بندوں پر مہربان ہے اس لیے یہ سلسلہ جاری رکھا گیا۔  ...
+ابن جر یراپنی تاریخ میں لکھتے ہیں کہ جب طالوت و جالوت کی جنگ ہوئی تو پہلے جالوت نے طالوت کو مقابلے کے لیے للکارا تھا اس لیے وہ اس کے مقابلے کے لیے آگے بڑھا تھا اور اپنے حریف جالوت کو قتل کر دیا تھا اور اس کے بعد طالوت اپنے عوام
```
غلطی: "ابن جر یر"
 درست: "ابن جریر" (درمیان میں غیر ضروری اسپیس)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+
+******
+
+[2^] ہمارے پیش نظر نسخے میں یہی لکھا ہے لیکن ابن جریر کے نسخہ تاریخ میں داؤد بن الشیی بن عوید  بن باغر بن سلمون بن نحشون ابن عمی نادب بن رام الخ ہے جب کہ نسخہ عرائس میں ان کا بسب نامہ کچھ اور دیا گیا ہے جس سے رجوع کیجئے ( محمود الامام ) نسخہ عرائس دستیاب نہ ہو سکا۔ (مترجم)
```
غلطی: "بسب نامہ"
 درست: "نسب نامہ" (صحیح لفظ "نسب نامہ" ہے، "بسب" غلط املا ہے)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+******
+
+ میں اتنے ہر دلعزیز ہوئے کہ انہوں نے انہیں اپنا بادشاہ تسلیم کر لیا۔
+کہا جاتا ہے کہ اس واقعے سے قبل شمویلؑ بنی اسرائیل کے بادشاہ تھے اور انہیں نے طالوت کو اپنا وارث و جانشین بنایا تھا۔ تا ہم ابن جریر کے بقول طالوت' جالوت کے قتل کے بعد لوگوں میں پسندیدگی و ہر دلعزیزی کی بناء پر بادشاہ ہوا تھا۔ واللہ اعلم
```
1۔ غلطی: "انہیں نے"
 درست: "انہی نے"

2۔ غلطی: "تا ہم"
درست: "تاہم" (یہ ایک ہی لفظ ہے، درمیان میں اسپیس نہیں آنی چاہیے)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+
+ میں اتنے ہر دلعزیز ہوئے کہ انہوں نے انہیں اپنا بادشاہ تسلیم کر لیا۔
+کہا جاتا ہے کہ اس واقعے سے قبل شمویلؑ بنی اسرائیل کے بادشاہ تھے اور انہیں نے طالوت کو اپنا وارث و جانشین بنایا تھا۔ تا ہم ابن جریر کے بقول طالوت' جالوت کے قتل کے بعد لوگوں میں پسندیدگی و ہر دلعزیزی کی بناء پر بادشاہ ہوا تھا۔ واللہ اعلم
+جیسا کہ پہلے بیان کیا گیا ہے کہ طالوت نے قصرام حکیم کے پاس جالوت کو قتل کیا تھا اس کا محل و قوع بقول ابن عسا کر وہی دریائے اردن کا کنارہ تھا جس کا قرآن پاک میں ذکر آیا ہے۔
```
غلطی: "ابن عسا کر"
 درست: "ابن عساکر" (صحیح نام "ابن عساکر" ہے، درمیان میں غلط اسپیس دی گئی ہے)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+ میں اتنے ہر دلعزیز ہوئے کہ انہوں نے انہیں اپنا بادشاہ تسلیم کر لیا۔
+کہا جاتا ہے کہ اس واقعے سے قبل شمویلؑ بنی اسرائیل کے بادشاہ تھے اور انہیں نے طالوت کو اپنا وارث و جانشین بنایا تھا۔ تا ہم ابن جریر کے بقول طالوت' جالوت کے قتل کے بعد لوگوں میں پسندیدگی و ہر دلعزیزی کی بناء پر بادشاہ ہوا تھا۔ واللہ اعلم
+جیسا کہ پہلے بیان کیا گیا ہے کہ طالوت نے قصرام حکیم کے پاس جالوت کو قتل کیا تھا اس کا محل و قوع بقول ابن عسا کر وہی دریائے اردن کا کنارہ تھا جس کا قرآن پاک میں ذکر آیا ہے۔
+طالوت کے ہاتھوں جالوت کے قتل کا ذکر فرمانے کے بعد اللہ تعالیٰ نے بنی اسرائیل کے قصے کے ضمن میں قرآن شریف میں یہ بھی ارشاد فرمایا کہ اس نے واؤوؑ کو اپنے فضل سے یہ معجزہ عطا فرمایا کہ ان کے ہاتھ میں آ کر لو ہا موم بن جاتا تھا جس سے وہ پہننے کے عام لباس کے علاوہ زرہ بھی بنا لیتے تھے جس سے وہ دشمنوں کے مقابلے میں زیادہ محفوظ ہو جاتے تھے۔ تا ہم ان کے (بنی اسرائیل کے) جو اعمال تھے اس سے اللہ تعالیٰ خوب ...
```
1۔ غلطی: "واؤوؑ"
 درست: "داؤدؑ" (یہاں واضح املا کی غلطی ہے)

2۔ غلطی: "لو ہا"
درست: "لوہا" (غلط اسپیس دی گئی ہے)

3۔ غلطی: "تا ہم"
درست: "تاہم" (درمیان میں اسپیس نہیں آتی)

4۔ غلطی: "داؤوؑ کے لیے"
درست: "داؤدؑ کے لیے" (یہاں بھی "داؤد" کی املا غلط ہے)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+جیسا کہ پہلے بیان کیا گیا ہے کہ طالوت نے قصرام حکیم کے پاس جالوت کو قتل کیا تھا اس کا محل و قوع بقول ابن عسا کر وہی دریائے اردن کا کنارہ تھا جس کا قرآن پاک میں ذکر آیا ہے۔
+طالوت کے ہاتھوں جالوت کے قتل کا ذکر فرمانے کے بعد اللہ تعالیٰ نے بنی اسرائیل کے قصے کے ضمن میں قرآن شریف میں یہ بھی ارشاد فرمایا کہ اس نے واؤوؑ کو اپنے فضل سے یہ معجزہ عطا فرمایا کہ ان کے ہاتھ میں آ کر لو ہا موم بن جاتا تھا جس سے وہ پہننے کے عام لباس کے علاوہ زرہ بھی بنا لیتے تھے جس سے وہ دشمنوں کے مقابلے میں زیادہ محفوظ ہو جاتے تھے۔ تا ہم ان کے (بنی اسرائیل کے) جو اعمال تھے اس سے اللہ تعالیٰ خوب ...
+یہ قصہ مجاہد و قتادہ اور حکم و عکرمہ نے بھی قرآن کے حوالے سے بہ تفصیل بیان کیا ہے۔
+حسن بصری، قتادہ اور اعمش بتاتے ہیں کہ داؤدؑ کے اس معجزے کی بناء پر جو انہیں اللہ تعالیٰ نے عطا فر مایا تھا انہیں لوہے کو تپانے اور کوٹنےپیٹنے کی ضرورت نہیں پڑتی تھی جیسے دوسروں کو پڑتی تھی اور وہ آسانی سے اس کے تار بنا کر یا اسے کڑیوں میں تبدیل کر کے بالترتیب اس سے لباس یا زرہ بنا لیتے تھے جس سے انہیں چھ ہزار درہم تک آمدنی ہو جاتی تھی۔
```
1۔ غلطی: "عطا فر مایا"
 درست: "عطا فرمایا" (غیر ضروری اسپیس)

2۔ غلطی: "کوٹنےپیٹنے"
 درست: "کوٹنے پیٹنے" (درمیان میں اسپیس ضروری ہے)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+داؤدؑ کا ذکر فرماتے ہوئے آنحضرتﷺ  نے بھی اپنی امت کو خود محنت کر کے حلال کی روزی حاصل کرنے کا حکم دیا ہے۔
+ابن عباس رضی اللہ عنہ اور مجاہد اور ان کے علاوہ قتادہ بھی کہتے ہیں کہ علماء وفقہاء اسلام کے ان افکار و اذ کار کا ماحصل یہ ہے کہ اسی اکل حلال سے اہل ایمان کو  وہ قوت حاصل ہوتی ہے کہ وہ رات بھر عبادت پر وردگار میں مشغول رہ کر نصف النہار تک روزہ سے رہ سکیں۔
+ صحیحین (صحیح مسلم وصحیح بخاری) میں آنحضرت ﷺ کا یہ ارشاد مذکور ہے کہ آپ نے فرمایا کہ ”میں اللہ تعالیٰ کی عبادت اس طرح پسند کرتا ہوں جیسے اس کی عبادت داؤد علیہ السلام کیا کرتے تھے اور روزہ بھی مجھے داؤد علیہ السلام کے روزہ کی طرح مرغوب ہے“۔
+حضرت داؤدؑ آ دھی رات تک سوتے اور پھر اس کے تہائی حصے میں عبادت الہٰی میں گزارنے کے بعد اس کے چھٹے حصے میں آرام کر لیا کرتے تھے۔ وہ ایک دن روزہ رکھتے اور ایک دن چھوڑ کر پھر روزہ رکھتے تھے۔
```
غلطی: "حضرت داؤدؑ آ دھی رات"
 درست: "حضرت داؤدؑ آدھی رات" (لفظ آدھی میں اسپیس ٹھیک کریں)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+اس کے علاوہ اللہ تعالیٰ نے داؤد علیہ السلام کو ایسی بے مثال آواز اور اس میں ایسا لحن عطا فرمایا تھا کہ ان کی قرأت سن کر پرندے ہوا میں اڑتے اڑتے رُک جاتے اور پہاڑ اور تمام چرند و پرندان کے ساتھ ذکر خداوندی میں مشغول ہوجاتے تھے۔
+اوز اعی نے عبداللہ بن عامر کے حوالے سے لحن داؤدی کے اس معجزے کا ذکر کیا ہے اور بتایا ہے کہ ان کی آواز سن کر وحوش و طیور ان کے گرد جمع ہو کر رقص کرنے لگتے اور رقص کرتے کرتے ان میں سے بعض بے ہوش اور بعض مر بھی جاتے تھے۔
+ وہب بن منبہ کہتے ہیں کہ داؤدؑ جب زبور کی تلاوت کرتے تو ان کی بے مثال آواز سن کر نہ صرف جن و انس چلتے چلتے رُک جاتے بلکہ وحوش وطیور ان کے گرد جمع ہو کر رقص کرتے اور کبھی رقص کرتے کرتے بے ہوش ہو جاتے اور کبھی کبھی مر بھی جاتے تھے۔
+ابوعوانہ سے باسناد مروی ہے کہ داؤدؑ بربط بجا کر زبور کی تلاوت کیا کرتے تھے'  لیکن یہ روایت غریب ہے۔ تا ہم آنحضرت ﷺ کی ایک حدیث مبارکہ کے حوالے سے بیان کیا گیا ہے کہ آپ نے فرمایا کہ ”ابو موسیٰ اشعری رضی اللہ عنہ نے اپنی آواز آل واؤو کے مزامیر سے وراثت میں پائی ہے“۔ کیونکہ ابو موسیٰ اشعری رضی اللہ عنہ کی تلاوت قرآن پاک میں خوش الحانی بھی کسی معجزے سے کم نہ تھی۔ یہ حدیث نبوی شیخین کے حوالے سے صحیح م ...
```
1۔ غلطی: "واؤو"
 درست: "داؤد" (واؤو غلط املا ہے، درست لفظ "داؤد" ہے)

2۔ غلطی: "تا ہم"
 درست: "تاہم" (درمیان میں غیر ضروری اسپیس)

3۔ غلطی: "آل واؤو کے مزامیر"
 درست: "آل داؤد کے مزامیر"

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+ابوعوانہ سے باسناد مروی ہے کہ داؤدؑ بربط بجا کر زبور کی تلاوت کیا کرتے تھے'  لیکن یہ روایت غریب ہے۔ تا ہم آنحضرت ﷺ کی ایک حدیث مبارکہ کے حوالے سے بیان کیا گیا ہے کہ آپ نے فرمایا کہ ”ابو موسیٰ اشعری رضی اللہ عنہ نے اپنی آواز آل واؤو کے مزامیر سے وراثت میں پائی ہے“۔ کیونکہ ابو موسیٰ اشعری رضی اللہ عنہ کی تلاوت قرآن پاک میں خوش الحانی بھی کسی معجزے سے کم نہ تھی۔ یہ حدیث نبوی شیخین کے حوالے سے صحیح م ...
+امام احمدؒ نے بھی حماد بن سلمہ کی زبانی محمد بن عمر' ابی سلمہ اور ابو ہریرہ رضی اللہ عنہ کے حوالے سے مندرجہ بالا حدیث نبوی بیان کی ہے۔
+داؤدؑ کی تلاوت زبور میں جیسا کہ امام احمدؒ نے اپنی مسند میں ایک دوسری جگہ عبدالرزاق کی زبانی اور معمر، ہمام اور ابو ہریرہ رضی اللہ عنہ کے حوالے سے بیان کیا ہے ایک تو کلام الہٰی کے اثر دوسرے خود ان کی ساحرانہ آواز کی تاثیر سے جو نتائج پیدا ہوتے تھے وہ بعید از قیاس نہیں ہیں۔
+بخاریؒ نے جو حدیث نبوی اس سلسلے میں خصوصیت سے عبد اللہ بن محمد اور عبد الرزاق کے حوالے سے پیش کی ہے اس میں آنحضرت ﷺ کا یہ ارشاد گرامی بیان کیا گیا ہے کہ داؤدؑ پہلے قرآن (اس حدیث میں جیسا کہ ظاہر ہے قرآن سے مراد ز بور ہے) کی تلاوت شروع کرتے اور پھر اپنا ساز چھیڑ تے تھے۔
```
غلطی: "چھیڑ تے" (الفاظ کے درمیان غیر ضروری اسپیس)
درست: "چھیڑتے"

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+اسی حدیث میں بخاریؒ نے آنحضرت ﷺ کے ارشاد گرامی کے حوالے سے داؤدؑ کے بدست خود  روزی کے حصول اور اکل حلال کا ذکر بھی کیا ہے۔
+ بخاریؒ کی بیان کردہ اس حدیث نبوی کو موسیٰ ابن عقبہ نے بھی صفوان یعنی ابن سلیم عطاء بن یسار اور ابو ہریرہ رضی اللہ عنہ کے حوالے سے روایت کیا ہے جب کہ ابن عسا کرنے اپنی تاریخ میں داؤدؑکے قصے کے ضمن میں ابراہیم بن طہمان'  موسیٰ بن عقبہ'  ابی عاصم' ابی بکر سبری اور صفوان بن سلیم کے حوالے سے داؤدؑ کا مذکورہ معجزہ یعنی ان کے ہاتھ میں آکر لو ہا موم ہو جاتا تھا اور ان کی بے مثال آواز کا ذکر کرتے ہوئے مذ ...
+قرآن شریف میں داؤدؑ کے قصے کے ضمن میں کلام الہٰی کے الفاظ ”(وَ آتَيْنَا داؤدَ زَبُورََا)“ (ترجمہ: اور ہم نے داؤد کو زبور دی)  جن کی تفسیر امام احمدؒ نے کی ہے زبور کے بارے میں بیان کیا ہے کہ وہ ماہ رمضان میں ان پر نازل ہوئی تھی اور وہ مواعظ و احکام پر مشتمل تھی لیکن اب وہ فی الجملہ محل نظر ہے کیونکہ اس میں متعدد مقامات پر اہل کتاب نے تحریف کر دی ہے۔
+جہاں تک قصہ داؤدؑ کے ضمن میں آیت قرآن(وَشَدَدْنَا مُلكَہُ وَ آتَيْنَا الْحِكْمَةَ وَفَضلَ الخطابِ) کا تعلق ہے اس کا بدیہی مطلب یہ ہے کہ اللہ تعالیٰ نے داؤدؑ کو ایک عظیم مملکت کی بادشاہت بخش تھی اور اس میں ان کے احکا م نافذ فرما دیے تھے۔
```
1۔ غلطی: "بخش تھی"
درست: "بخشی تھی"

2۔ غلطی: "احکا م"
درست: "احکام" (درمیان میں غیر ضروری اسپیس)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+وہ شخص بولا: ”یا نبی اللہ! مجھے اس شخص کے باپ پر میری گائے غصب کرنے کا علم ہوا تھا لیکن میں نے دعویٰ بیٹے پر کر دیا جس کے لیے میں معافی چاہتا ہوں“۔
+داؤدؑ نے اس شخص یعنی مدعی کو غلط دعویٰ کرنے کی پاداش میں قتل کر دیا تو لوگوں کا ان کے فیصلوں میں سختی سے عدل و انصاف پر عمل پیرا ہونے کا اور زیادہ یقین ہو گیا اور اس طرح داؤدؑ کی دھاک بیٹھ گئی اور وہ ان کی صدق دل سے فرمانبرداری کرنے لگے۔
+ابن عباس رضی اللہ عنہ کہتے ہیں کہ قرآن شریف میں اللہ تعالیٰ کے ارشاد ”شَدَدْنَا مُلْكَہُ“ (ترجمہ: ہم نے اس کی سلطنت کو مضبوط کیا)  سے اسی واقعے کی طرف اشارہ ہے اور ”آتَيْنَاهُ الحِكْمَةَ“ (ترجمہ: ہم نے ان کو حکمت عطا کی) کا مطلب نبوت ہے۔
+جہاں تک قرآن پاک میں ارشاد باری تعالیٰ ”وَ فضلَ الْخِطَاب“ (ترجمہ: اورفضیلت دی  کلام سے) کا تعلق ہے اس کے بارے میں شریح وشعمی اور قتادہ و ابو عبد الرحمٰن سلمی کہتے ہیں کہ اس کا مطلب شہو د و ایمان ہے جس کی بناء پر داؤدؑ نے مدعی کو قتل کر دیا اور مدعا علیہ کو چھوڑ دیا تھا۔
```
1۔ غلطی: "اورفضیلت"
درست: "اور فضیلت" (درمیان میں اسپیس کی کمی)

2۔ غلطی: "شعمی"
درست: "شعبی" (یعنی "شعمی" کی جگہ "شعبی" درست ہے)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+ابن عباس رضی اللہ عنہ کہتے ہیں کہ قرآن شریف میں اللہ تعالیٰ کے ارشاد ”شَدَدْنَا مُلْكَہُ“ (ترجمہ: ہم نے اس کی سلطنت کو مضبوط کیا)  سے اسی واقعے کی طرف اشارہ ہے اور ”آتَيْنَاهُ الحِكْمَةَ“ (ترجمہ: ہم نے ان کو حکمت عطا کی) کا مطلب نبوت ہے۔
+جہاں تک قرآن پاک میں ارشاد باری تعالیٰ ”وَ فضلَ الْخِطَاب“ (ترجمہ: اورفضیلت دی  کلام سے) کا تعلق ہے اس کے بارے میں شریح وشعمی اور قتادہ و ابو عبد الرحمٰن سلمی کہتے ہیں کہ اس کا مطلب شہو د و ایمان ہے جس کی بناء پر داؤدؑ نے مدعی کو قتل کر دیا اور مدعا علیہ کو چھوڑ دیا تھا۔
+ مجاہد والسدی کہتے ہیں کہ داؤدؑ کے اس فیصلے سے ان کی اصابت قضا یعنی فیصلے اور فہم و فراست کا ثبوت ملتا ہے۔ مجاہد یہ بھی کہتے ہیں کہ ”فصل الخطاب“ میں داؤدؑ کے کلام اور ان کے فیصلے کے مابین فصل کی طرف اشارہ ہے۔ ابن جریر نے مجاہد کی اس رائے سے اتفاق کیا ہے اور مجاہد کی یہ رائے ابو موسیٰ کے اس قول کے منافی بھی نہیں ہے جس میں انہوں نے ”اما بعد“ کہا ہے۔
+ وہب بن منبہ کہتے ہیں کہ جب بنی اسرائیل میں باہمی جھگڑوں اور گواہیوں کی کثرت ہوئی تو داؤدؑ کو ان کی سماعت اور پھر فیصلہ کرنے میں جو فصل غور کرنے اور کسی حتمی نتیجے تک پہنچنے میں رکھنا پڑتا تھا کلام الہٰی میں ”فصل الخطاب“ سے و ہی مراد ہے۔ اس کے علاوہ ان کے اکثر فیصلوں میں ہدایت خداوندی بھی شامل ہوتی تھی اور مقدمات کا سماعت کے لیے ان کے سامنےپیش ہوئے اور اللہ تعالیٰ کی طرف سے حکم موصول ہونے میں جو  ...
```
غلطی: "سامنےپیش"
درست: "سامنے پیش" (درمیان میں اسپیس کی کمی)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+جہاں تک قرآن پاک میں ارشاد باری تعالیٰ ”وَ فضلَ الْخِطَاب“ (ترجمہ: اورفضیلت دی  کلام سے) کا تعلق ہے اس کے بارے میں شریح وشعمی اور قتادہ و ابو عبد الرحمٰن سلمی کہتے ہیں کہ اس کا مطلب شہو د و ایمان ہے جس کی بناء پر داؤدؑ نے مدعی کو قتل کر دیا اور مدعا علیہ کو چھوڑ دیا تھا۔
+ مجاہد والسدی کہتے ہیں کہ داؤدؑ کے اس فیصلے سے ان کی اصابت قضا یعنی فیصلے اور فہم و فراست کا ثبوت ملتا ہے۔ مجاہد یہ بھی کہتے ہیں کہ ”فصل الخطاب“ میں داؤدؑ کے کلام اور ان کے فیصلے کے مابین فصل کی طرف اشارہ ہے۔ ابن جریر نے مجاہد کی اس رائے سے اتفاق کیا ہے اور مجاہد کی یہ رائے ابو موسیٰ کے اس قول کے منافی بھی نہیں ہے جس میں انہوں نے ”اما بعد“ کہا ہے۔
+ وہب بن منبہ کہتے ہیں کہ جب بنی اسرائیل میں باہمی جھگڑوں اور گواہیوں کی کثرت ہوئی تو داؤدؑ کو ان کی سماعت اور پھر فیصلہ کرنے میں جو فصل غور کرنے اور کسی حتمی نتیجے تک پہنچنے میں رکھنا پڑتا تھا کلام الہٰی میں ”فصل الخطاب“ سے و ہی مراد ہے۔ اس کے علاوہ ان کے اکثر فیصلوں میں ہدایت خداوندی بھی شامل ہوتی تھی اور مقدمات کا سماعت کے لیے ان کے سامنےپیش ہوئے اور اللہ تعالیٰ کی طرف سے حکم موصول ہونے میں جو  ...
+ واؤدؑ کے بارے میں جو قصے بیان کیے گئے ہیں ان میں اسرائیلیات پر مبنی حکایات کثرت سے شامل کی گئی ہیں۔ اس لیے ہم نے ان کے بارے میں صرف اس قصے پر اکتفا کی ہے جو قرآن شریف میں خود اللہ تعالیٰ نے بیان فرمایا ہے۔ اللہ تعالیٰ جسے چاہتا ہے صراط مستقیم پر چلنے کی ہدایت فرماتا ہے:
```
غلطی: "واؤدؑ"
درست: "داؤدؑ" (غلط املا)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+ وہب بن منبہ کہتے ہیں کہ جب بنی اسرائیل میں باہمی جھگڑوں اور گواہیوں کی کثرت ہوئی تو داؤدؑ کو ان کی سماعت اور پھر فیصلہ کرنے میں جو فصل غور کرنے اور کسی حتمی نتیجے تک پہنچنے میں رکھنا پڑتا تھا کلام الہٰی میں ”فصل الخطاب“ سے و ہی مراد ہے۔ اس کے علاوہ ان کے اکثر فیصلوں میں ہدایت خداوندی بھی شامل ہوتی تھی اور مقدمات کا سماعت کے لیے ان کے سامنےپیش ہوئے اور اللہ تعالیٰ کی طرف سے حکم موصول ہونے میں جو  ...
+ واؤدؑ کے بارے میں جو قصے بیان کیے گئے ہیں ان میں اسرائیلیات پر مبنی حکایات کثرت سے شامل کی گئی ہیں۔ اس لیے ہم نے ان کے بارے میں صرف اس قصے پر اکتفا کی ہے جو قرآن شریف میں خود اللہ تعالیٰ نے بیان فرمایا ہے۔ اللہ تعالیٰ جسے چاہتا ہے صراط مستقیم پر چلنے کی ہدایت فرماتا ہے:
+”وَاللهُ يَهْدِى مَنْ يَشَاءُ إِلَى صِرَاطٍ مُّسْتَقِيم“ (ترجمہ: اور اللہ ہدایت دیتا ہے جسے چاہتا ہے سیدھے راستے کی طرف)
+کسی شخص نے ابن عباس رضی اللہ عنہ سے داؤدؑ کا قصہ سنانے کی درخواست کی تو انہوں نے جوا باََ فرمایا کہ
```
غلطی: "جوا باََ"
درست: "جواباً" (درست املا)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+ واؤدؑ کے بارے میں جو قصے بیان کیے گئے ہیں ان میں اسرائیلیات پر مبنی حکایات کثرت سے شامل کی گئی ہیں۔ اس لیے ہم نے ان کے بارے میں صرف اس قصے پر اکتفا کی ہے جو قرآن شریف میں خود اللہ تعالیٰ نے بیان فرمایا ہے۔ اللہ تعالیٰ جسے چاہتا ہے صراط مستقیم پر چلنے کی ہدایت فرماتا ہے:
+”وَاللهُ يَهْدِى مَنْ يَشَاءُ إِلَى صِرَاطٍ مُّسْتَقِيم“ (ترجمہ: اور اللہ ہدایت دیتا ہے جسے چاہتا ہے سیدھے راستے کی طرف)
+کسی شخص نے ابن عباس رضی اللہ عنہ سے داؤدؑ کا قصہ سنانے کی درخواست کی تو انہوں نے جوا باََ فرمایا کہ
+”یوں تو ان کے بارے میں بے شمار قصے مشہور ہیں جن میں سے اکثر و بیشتر اسرائیلی حکایات پر مبنی ہیں لیکن ان کا سچا قصہ اللہ تعالیٰ نے قرآن شریف میں بیان فرمایا ہے۔ اس کے علاوہ ان کے بارے میں رسول اللہ ﷺ کی ایک حدیث یہ ہے کہ ”افضل ترین روزہ دادؤدؑ کا روزہ ہے اور یہ کہ وہ نماز میں زبور کی تلاوت ستر طریقے سے ٹھہر ٹھہر کر اور خوش الحانی سے کیا کرتے تھے جس کے دوران میں ان پر رقت و گر یہ طاری رہتا تھا اور  ...
```
1۔ فقرہ: "دادؤدؑ کا روزہ"
غلطی: "دادؤدؑ"
درست: "داؤدؑ" (غلط املا)

2۔ فقرہ: "جس کے دوران میں ان پر رقت و گر یہ طاری رہتا تھا"
غلطی: "گر یہ"
درست: "گریہ" (اسپیس اضافی ہے)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+کسی شخص نے ابن عباس رضی اللہ عنہ سے داؤدؑ کا قصہ سنانے کی درخواست کی تو انہوں نے جوا باََ فرمایا کہ
+”یوں تو ان کے بارے میں بے شمار قصے مشہور ہیں جن میں سے اکثر و بیشتر اسرائیلی حکایات پر مبنی ہیں لیکن ان کا سچا قصہ اللہ تعالیٰ نے قرآن شریف میں بیان فرمایا ہے۔ اس کے علاوہ ان کے بارے میں رسول اللہ ﷺ کی ایک حدیث یہ ہے کہ ”افضل ترین روزہ دادؤدؑ کا روزہ ہے اور یہ کہ وہ نماز میں زبور کی تلاوت ستر طریقے سے ٹھہر ٹھہر کر اور خوش الحانی سے کیا کرتے تھے جس کے دوران میں ان پر رقت و گر یہ طاری رہتا تھا اور  ...
+ اس کے بعد ابن عباس رضی اللہ عنہ نے اس شخص سے فرمایا:
+”ویسے اگر تم چاہو تو میں تمہیں یہ بتا سکتا ہوں کہ داؤدؑ کے بیٹے سلیمانؑ ہر مہینے کے پہلے تین دن پھر، اس کے وسط میں تین دن اور اس کے آخر میں تین دن روزہ رکھا کرتے تھے اور وہ جب کسی شہر کو فتح کرنا چاہتے تھے تو اس وقت ظاہر ہے کہ خواہ مہینے کے پہلے تین دن ہوں، اس کے درمیانی تین دن ہوں یا آخری تین دن وہ ہمیشہ روزہ سے ہوتے تھے۔ میں تمہیں یہ بھی بتا سکتا ہوں عیسٰی بن مریم علیہ السلام دن کے وقت ہمیشہ رو ...
```
"اور بیچ میں دو دن کا نا غہ کر کے پھر روزہ رکھا کرتی تھیں"
غلطی: "نا غہ"
درست: "ناغہ" اسپیس remove کریں)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+ اس کے بعد ابن عباس رضی اللہ عنہ نے اس شخص سے فرمایا:
+”ویسے اگر تم چاہو تو میں تمہیں یہ بتا سکتا ہوں کہ داؤدؑ کے بیٹے سلیمانؑ ہر مہینے کے پہلے تین دن پھر، اس کے وسط میں تین دن اور اس کے آخر میں تین دن روزہ رکھا کرتے تھے اور وہ جب کسی شہر کو فتح کرنا چاہتے تھے تو اس وقت ظاہر ہے کہ خواہ مہینے کے پہلے تین دن ہوں، اس کے درمیانی تین دن ہوں یا آخری تین دن وہ ہمیشہ روزہ سے ہوتے تھے۔ میں تمہیں یہ بھی بتا سکتا ہوں عیسٰی بن مریم علیہ السلام دن کے وقت ہمیشہ رو ...
+اس کے بعد ابن عباس رضی اللہ عنہ نے اس شخص سے فرمایا:
+”اگر تم چاہو تو میں تمہیں یہ بھی بتا سکتا ہوں کہ رسول عربی الامی حضرت محمد ﷺ ہر مہینے کے تین دن روزہ رکھاکرتے تھے“۔
```
"روزہ رکھاکرتے تھے"
غلطی: "رکھاکرتے"
درست: "رکھا کرتے" (الفاظ کے درمیان اسپیس درکار ہے)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+
+## داؤد علیہ السلام کی کمیت حیات و کیفیت وفات
+
+تخلیق آدمؑ کے بارے میں احادیث نبوی سے حوالے پیش کیے جاچکے ہیں۔ جب اللہ تعالیٰ نے انہیں اور حوا کو جنت سے نکال کر زمین پر جانے کا حکم دیا' تو اس کے بعد حوا کے بطن سے آدم ؑ کی جو اولاد پیدا ہوئی وہ رفتہ رفتہ یکے بعد دیگرے وفات پاتی چلی گئی جن میں انبیاء علیہمالسلام بھی تھے۔ آخر اپنی اولاد میں ایک ممتاز شخصیت کو سامنے دیکھ کر آدمؑ نے اللہ تعالیٰ سے سوال کیا: ”یہ کون ہے؟“ جواب ملا: ”تمہارا بیٹا داؤدؑ ...
```
غلطی: "علیہمالسلام"
درست: "علیہما السلام" (درست املا)

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+آدم ؑ نے اپنے بیٹے کی شان و شوکت' نبوت و بادشاہت کا اعزاز اور بنی اسرائیل میں اس کی عزت وحرمت اورپسندیدگی و ہر دلعزیزی دیکھ کر اللہ تعالیٰ سے عرض کیا:
+”اس کی ساری عمر کتنی ہو گی؟“۔
+جواب ملا: ”ساٹھ سال“۔
+آدم ؑنے عرض کیا:”یا رب! اس کی عمر میں اضافہ فرما دے“۔
```
غلطی: "آدم ؑنے" (”ؑ“ اور "نے" کے درمیان فاصلہ نہیں)
درست: "آدم ؑ نے عرض کیا:

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+جواب ملا: ”ساٹھ سال“۔
+آدم ؑنے عرض کیا:”یا رب! اس کی عمر میں اضافہ فرما دے“۔
+جواب ملا: ”اس کی عمر میں اضافے کی واحد صورت یہ ہے کہ تمہاری باقی عمر سے دے دی جائے“۔ 
+آدمؑ نے خدا کے سامنے رضامندی کا اظہار کیا تو اس نے آدمؑ کی عمر سے جو ایک ہزار چالیس سال ہو نا تھی چالیس سال نکال کر ان کے اس بیٹے یعنی داؤدؑ کی عمر میں شامل کر دیئے تو اس کی عمر سو سال ہوگئی لیکن خود آدمؑ کے کی عمر جو ایک ہزار چالیس سال ہو نا تھی اب صرف ایک ہزار سال رہ گئی جو اس دُعا کے وقت ان کی عمر تھی۔ تا ہم جب آدمؑ کی وفات کا وقت آیا تو وہ یہ بات بھول چکے تھے کہ انہوں نے اپنی عمر کے چالیس سال ...
```
1۔ غلطی: "آدمؑ کے کی عمر" (غیر ضروری "کے" شامل ہے)
درست: "آدمؑ کی عمر جو ایک ہزار چالیس سال ہونا تھی"

2۔ غلطی: "ہو نا" (غلط املا، اضافی فاصلہ)
درست: "ہونا تھی"

3۔ غلطی: "تا ہم" (غلط املا، غیر ضروری فاصلہ)
درست: "تاہم جب آدمؑ کی وفات کا وقت آیا تو وہ یہ بات بھول چکے تھے"


4۔ "کیونکہ یہ بات آئند ہ صدیوں بعد کے واقعات سے متعلق تھی"
غلطی: "آئند ہ" (غلط املا، اضافی فاصلہ)
درست: "آئندہ"

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+آدم ؑنے عرض کیا:”یا رب! اس کی عمر میں اضافہ فرما دے“۔
+جواب ملا: ”اس کی عمر میں اضافے کی واحد صورت یہ ہے کہ تمہاری باقی عمر سے دے دی جائے“۔ 
+آدمؑ نے خدا کے سامنے رضامندی کا اظہار کیا تو اس نے آدمؑ کی عمر سے جو ایک ہزار چالیس سال ہو نا تھی چالیس سال نکال کر ان کے اس بیٹے یعنی داؤدؑ کی عمر میں شامل کر دیئے تو اس کی عمر سو سال ہوگئی لیکن خود آدمؑ کے کی عمر جو ایک ہزار چالیس سال ہو نا تھی اب صرف ایک ہزار سال رہ گئی جو اس دُعا کے وقت ان کی عمر تھی۔ تا ہم جب آدمؑ کی وفات کا وقت آیا تو وہ یہ بات بھول چکے تھے کہ انہوں نے اپنی عمر کے چالیس سال ...
+یہ ر وایت امام احمدؒ نے ابن عباس رضی اللہ عنہ 'ترمذیؒ' کی صحیح 'ابو ہریرہ رضی اللہ عنہ ابن خزیمہ اور ابن حبان کے حوالے سے بیان کی ہے۔ حاکم نے اس روایت کو مسلمؒ کی سند کی طرح بیان کیا ہے۔ حاکم کا طرز بیان ہم اس سے قبل قصہ آدمؑ میں واضح کر چکے ہیں۔ بعض اہل کتاب نے داؤدؑ کی عمرسڑ سٹھ سال بیان کی ہے اور ان کے دور حکومت کو چالیس سال بتایا ہے جب کہ ان کی عمر  کے بارے میں ان کا اول الذ کر بیان مندرجہ با ...
```
1۔ "یہ ر وایت امام احمدؒ نے ابن عباس رضی اللہ عنہ 'ترمذیؒ' کی صحیح 'ابو ہریرہ رضی اللہ عنہ ابن خزیمہ اور ابن حبان کے حوالے سے بیان کی ہے۔"
غلطی: "یہ ر وایت" (اضافی اسپیس)
درست: "یہ روایت"

2۔ "عمرسڑ سٹھ سال"
غلطی: "عمرسڑ سٹھ" (اسپیکنگ issue)
درست: "عمر سڑسٹھ سال"

3۔ "جب کہ ان کی عمر کے بارے میں ان کا اول الذ کر بیان"
غلطی: "اول الذ کر" (صحیح املا "اول اذکر")

4۔ "قطعاََ نا قابل قبول ہے۔"
غلطی: "نا قابل" (صحیح املا "ناقابل")
درست: "قطعاََ ناقابل قبول ہے۔"

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+جواب ملا: ”اس کی عمر میں اضافے کی واحد صورت یہ ہے کہ تمہاری باقی عمر سے دے دی جائے“۔ 
+آدمؑ نے خدا کے سامنے رضامندی کا اظہار کیا تو اس نے آدمؑ کی عمر سے جو ایک ہزار چالیس سال ہو نا تھی چالیس سال نکال کر ان کے اس بیٹے یعنی داؤدؑ کی عمر میں شامل کر دیئے تو اس کی عمر سو سال ہوگئی لیکن خود آدمؑ کے کی عمر جو ایک ہزار چالیس سال ہو نا تھی اب صرف ایک ہزار سال رہ گئی جو اس دُعا کے وقت ان کی عمر تھی۔ تا ہم جب آدمؑ کی وفات کا وقت آیا تو وہ یہ بات بھول چکے تھے کہ انہوں نے اپنی عمر کے چالیس سال ...
+یہ ر وایت امام احمدؒ نے ابن عباس رضی اللہ عنہ 'ترمذیؒ' کی صحیح 'ابو ہریرہ رضی اللہ عنہ ابن خزیمہ اور ابن حبان کے حوالے سے بیان کی ہے۔ حاکم نے اس روایت کو مسلمؒ کی سند کی طرح بیان کیا ہے۔ حاکم کا طرز بیان ہم اس سے قبل قصہ آدمؑ میں واضح کر چکے ہیں۔ بعض اہل کتاب نے داؤدؑ کی عمرسڑ سٹھ سال بیان کی ہے اور ان کے دور حکومت کو چالیس سال بتایا ہے جب کہ ان کی عمر  کے بارے میں ان کا اول الذ کر بیان مندرجہ با ...
+مندرجہ بالا روایت کے مطابق داؤدؑ کی تجہیز و تکفین کے بعد ان کی میت جہاں رکھی گئی تھی وہاں دھوپ تھی۔ یہ دیکھ کراُن کے بیٹے سلیمانؑ نے بڑے پرندے کو حکم دیا کہ وہ ان کے باپ کی میت پر اپنے پنکھ پھیلا کر سایہ کردے۔
```
1۔ غلطی: "یہ دیکھ کراُن" (اسپیکنگ ٹھیک نہیں)
درست: "یہ دیکھ کر اُن کے بیٹے"

2۔ "بڑے پرندے کو حکم دیا کہ وہ ان کے باپ کی میت پر اپنے پنکھ پھیلا کر سایہ کردے۔"
غلطی: "کردے" (صحیح اسپیکنگ کے ساتھ "کر دے")
درست: "بڑے پرندے کو حکم دیا کہ وہ ان کے باپ کی میت پر اپنے پنکھ پھیلا کر سایہ کر دے۔"

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+یہ ر وایت امام احمدؒ نے ابن عباس رضی اللہ عنہ 'ترمذیؒ' کی صحیح 'ابو ہریرہ رضی اللہ عنہ ابن خزیمہ اور ابن حبان کے حوالے سے بیان کی ہے۔ حاکم نے اس روایت کو مسلمؒ کی سند کی طرح بیان کیا ہے۔ حاکم کا طرز بیان ہم اس سے قبل قصہ آدمؑ میں واضح کر چکے ہیں۔ بعض اہل کتاب نے داؤدؑ کی عمرسڑ سٹھ سال بیان کی ہے اور ان کے دور حکومت کو چالیس سال بتایا ہے جب کہ ان کی عمر  کے بارے میں ان کا اول الذ کر بیان مندرجہ با ...
+مندرجہ بالا روایت کے مطابق داؤدؑ کی تجہیز و تکفین کے بعد ان کی میت جہاں رکھی گئی تھی وہاں دھوپ تھی۔ یہ دیکھ کراُن کے بیٹے سلیمانؑ نے بڑے پرندے کو حکم دیا کہ وہ ان کے باپ کی میت پر اپنے پنکھ پھیلا کر سایہ کردے۔
+امام احمدؒ یہ روایت بیان کرنے کے بعد ابو ہریرہ رضی اللہ عنہ کے حوالے سے بیان کرتے ہیں کہ ابو ہریرہ رضی اللہ عنہ نے رسول اللہﷺ کی وفات کا منظر بچشم خود دیکھ کر بتایا کہ ایک پرندہ کہیں سے آیا اپنا صرف ایک پنکھ آپ کے سینہ مبارک پر پھیلایا اور آپ کی روح پاک آسانی سے آنا ََفاناََ قبض کر کے نہ جانے کدھر سے باہر چلا گیا۔
+یہ روایت امام احمدؒ کی بیان کر دہ اور اُن کی اسناد انتہائی قوی ہیں اور ثقہ افراد پر مشتمل ہیں۔
```
"بیان کر دہ"
غلطی: "کر دہ" (یہ املا کی غلطی ہے)
درست: "کردہ"

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+مندرجہ بالا روایت کے مطابق داؤدؑ کی تجہیز و تکفین کے بعد ان کی میت جہاں رکھی گئی تھی وہاں دھوپ تھی۔ یہ دیکھ کراُن کے بیٹے سلیمانؑ نے بڑے پرندے کو حکم دیا کہ وہ ان کے باپ کی میت پر اپنے پنکھ پھیلا کر سایہ کردے۔
+امام احمدؒ یہ روایت بیان کرنے کے بعد ابو ہریرہ رضی اللہ عنہ کے حوالے سے بیان کرتے ہیں کہ ابو ہریرہ رضی اللہ عنہ نے رسول اللہﷺ کی وفات کا منظر بچشم خود دیکھ کر بتایا کہ ایک پرندہ کہیں سے آیا اپنا صرف ایک پنکھ آپ کے سینہ مبارک پر پھیلایا اور آپ کی روح پاک آسانی سے آنا ََفاناََ قبض کر کے نہ جانے کدھر سے باہر چلا گیا۔
+یہ روایت امام احمدؒ کی بیان کر دہ اور اُن کی اسناد انتہائی قوی ہیں اور ثقہ افراد پر مشتمل ہیں۔
+جوہری کی روایت کے مطابق جس پرندے نے داؤدؑ کی میت پر سلیمانؑ کے حکم سے سایہ کیا تھا وہ انتہائی لمبےپنکھوں والا شاہین تھا۔
```
"انتہائی لمبےپنکھوں والا"
غلطی: "لمبےپنکھوں" (الفاظ کے درمیان اسپیس نہیں)
درست: "لمبے پنکھوں والا"

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+امام احمدؒ یہ روایت بیان کرنے کے بعد ابو ہریرہ رضی اللہ عنہ کے حوالے سے بیان کرتے ہیں کہ ابو ہریرہ رضی اللہ عنہ نے رسول اللہﷺ کی وفات کا منظر بچشم خود دیکھ کر بتایا کہ ایک پرندہ کہیں سے آیا اپنا صرف ایک پنکھ آپ کے سینہ مبارک پر پھیلایا اور آپ کی روح پاک آسانی سے آنا ََفاناََ قبض کر کے نہ جانے کدھر سے باہر چلا گیا۔
+یہ روایت امام احمدؒ کی بیان کر دہ اور اُن کی اسناد انتہائی قوی ہیں اور ثقہ افراد پر مشتمل ہیں۔
+جوہری کی روایت کے مطابق جس پرندے نے داؤدؑ کی میت پر سلیمانؑ کے حکم سے سایہ کیا تھا وہ انتہائی لمبےپنکھوں والا شاہین تھا۔
+ السد ی ابی مالک اور ابن عباس رضی اللہ عنہ کے حوالے سے بیان کرتے ہیں کہ داؤدؑ کی وفات سنیچر کے روز صبح کے وقت ہوئی تھی اور ان کی میت پر ایک پرندے نے دھوپ کی وجہ سے سایہ کیا تھا جب کہ اسحاق بن بشر سعید بن ابی عروبہ اور قتادہ و حسن کے حوالے سے کہتے ہیں کہ ان کی وفات چہار شنبہ یعنی بدھ کے روز صبح کے وقت ہوئی تھی اور اس وقت ان کی عمر سو سال تھی۔
```
"السد ی ابی مالک اور ابن عباس رضی اللہ عنہ کے حوالے سے بیان کرتے ہیں"
غلطی: "السد ی" (الفاظ کے درمیان extra اسپیس)
درست: "السدی"

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+یہ روایت امام احمدؒ کی بیان کر دہ اور اُن کی اسناد انتہائی قوی ہیں اور ثقہ افراد پر مشتمل ہیں۔
+جوہری کی روایت کے مطابق جس پرندے نے داؤدؑ کی میت پر سلیمانؑ کے حکم سے سایہ کیا تھا وہ انتہائی لمبےپنکھوں والا شاہین تھا۔
+ السد ی ابی مالک اور ابن عباس رضی اللہ عنہ کے حوالے سے بیان کرتے ہیں کہ داؤدؑ کی وفات سنیچر کے روز صبح کے وقت ہوئی تھی اور ان کی میت پر ایک پرندے نے دھوپ کی وجہ سے سایہ کیا تھا جب کہ اسحاق بن بشر سعید بن ابی عروبہ اور قتادہ و حسن کے حوالے سے کہتے ہیں کہ ان کی وفات چہار شنبہ یعنی بدھ کے روز صبح کے وقت ہوئی تھی اور اس وقت ان کی عمر سو سال تھی۔
+ ابو السکن ہجری بیان کرتے ہیں کہ داؤدؑ ان کے بیٹے سلیمانؑ اور حضرت ابراہیمؑ  سب نے صبح کے وقت وفات پائی تھی۔ ابن عسا کر اپنی اسناد پیش کرتے ہوئے کہتے ہیں کہ داؤدؑ اپنی سجدہ گاہ سے نکل رہے تھے کہ ملک الموت ان کے سامنے حاضر ہو گیا۔ اسے دیکھ کر انہوں نے فرمایا کہ وہ کھڑے رہیں یا بیٹھ جائیں۔ یہ سن کر ملک الموت نے جواب دیا کہ یا نبی اللہ مجھے سنین و شہور اور آثار و اوزان پر مشتمل صدیوں کا حساب دینا ہو ...
```
1۔ "ابو السکن ہجری بیان کرتے ہیں"
غلطی: "ابو السکن" (الفاظ کے درمیان غیر ضروری اسپیس)
درست: "ابوالسکن"

2۔ "ابن عسا کر اپنی اسناد پیش کرتے ہوئے کہتے ہیں"
غلطی: "ابن عسا کر" (الفاظ کے درمیان غیر ضروری اسپیس)
درست: "ابن عساکر"

3۔ "ملک الموت سے یہ سن کر داؤدؑ جس چٹائی پر کھڑے تھے اس پر بیٹھ کر سجدے میں چلے گئے اور ان کی روح قفس عنصری سے پرواز کرگئی۔"
غلطی: "پرواز کرگئی" (الفاظ کے درمیان مناسب فاصلہ نہیں)
درست: "پرواز کر گئی"

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+جوہری کی روایت کے مطابق جس پرندے نے داؤدؑ کی میت پر سلیمانؑ کے حکم سے سایہ کیا تھا وہ انتہائی لمبےپنکھوں والا شاہین تھا۔
+ السد ی ابی مالک اور ابن عباس رضی اللہ عنہ کے حوالے سے بیان کرتے ہیں کہ داؤدؑ کی وفات سنیچر کے روز صبح کے وقت ہوئی تھی اور ان کی میت پر ایک پرندے نے دھوپ کی وجہ سے سایہ کیا تھا جب کہ اسحاق بن بشر سعید بن ابی عروبہ اور قتادہ و حسن کے حوالے سے کہتے ہیں کہ ان کی وفات چہار شنبہ یعنی بدھ کے روز صبح کے وقت ہوئی تھی اور اس وقت ان کی عمر سو سال تھی۔
+ ابو السکن ہجری بیان کرتے ہیں کہ داؤدؑ ان کے بیٹے سلیمانؑ اور حضرت ابراہیمؑ  سب نے صبح کے وقت وفات پائی تھی۔ ابن عسا کر اپنی اسناد پیش کرتے ہوئے کہتے ہیں کہ داؤدؑ اپنی سجدہ گاہ سے نکل رہے تھے کہ ملک الموت ان کے سامنے حاضر ہو گیا۔ اسے دیکھ کر انہوں نے فرمایا کہ وہ کھڑے رہیں یا بیٹھ جائیں۔ یہ سن کر ملک الموت نے جواب دیا کہ یا نبی اللہ مجھے سنین و شہور اور آثار و اوزان پر مشتمل صدیوں کا حساب دینا ہو ...
+اسحاق بن بشر کہتے ہیں کہ انہیں وافر بن سلیمان نے ابی سلیمان فلسطینی اور وہب بن منبہ کے حوالے سے بتایا کہ داؤدؑ کی وفات کے دن ان کے جنازے میں شرکت کرنے والے لوگ جن میں دوسرے لوگوں کے علاوہ صرف راہبوں کی تعداد چالیس ہزار تھی سب کے سب دھوپ میں بیٹھے تھے اور موسم بھی گرمی کا تھا۔ یہ دیکھ کر سلیمان علیہ السلام نے جنہیں اللہ تعالیٰ نے ابتداء ہی سے یہ اعجاز بخشا تھا کہ پرندوں کو حکم دیا کہ وہ حاضرین پر  ...
```
"اتنی دور ر ہیں کہ اس جگہ ہوا تھوڑی بہت آتی رہے"
غلطی: "دور ر ہیں" (الفاظ کے درمیان غیر ضروری اسپیس)
درست: "دور رہیں"

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+
+## قصہ سلیمان بن داؤد علیہ السلام
+
+حافظ ابن عساکر کے بقول سلیمانؑ کا پورا نام سلیمان بن داؤ دبن ایشا بن عويد بن عابر بن سلمون بن تحشون بن عمینا داب بن ارم بن حصرون بن فارص بن یہوذا بن یعقوب بن اسحٰق بن ابراہیم بن ابی الربیع نبی اللہ بن نبی اللہ علیہ السلام ایک عرصے سے مشہور چلا آتا ہے۔
```
1۔ غلطی: داؤ دبن (اسپیس)
درست: داؤد بن

2۔ غلطی: عمینا داب (اسپیس)
درست: عمیناداب

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+## قصہ سلیمان بن داؤد علیہ السلام
+
+حافظ ابن عساکر کے بقول سلیمانؑ کا پورا نام سلیمان بن داؤ دبن ایشا بن عويد بن عابر بن سلمون بن تحشون بن عمینا داب بن ارم بن حصرون بن فارص بن یہوذا بن یعقوب بن اسحٰق بن ابراہیم بن ابی الربیع نبی اللہ بن نبی اللہ علیہ السلام ایک عرصے سے مشہور چلا آتا ہے۔
+بعض کتابوں سے ظاہر ہوتا ہے کہ وہ یعنی سلیمانؑ د مشق گئے تھے۔ ابن ماکولا نے ان کا نسب نامہ جو بتایا ہے وہ وہی ہے جو ابن عساکر نے بتایا ہے۔
```
بعض کتابوں سے ظاہر ہوتا ہے کہ وہ یعنی سلیمانؑ د مشق گئے تھے۔

غلطی: د مشق (اسپیس)
درست: دمشق

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+
+حافظ ابن عساکر کے بقول سلیمانؑ کا پورا نام سلیمان بن داؤ دبن ایشا بن عويد بن عابر بن سلمون بن تحشون بن عمینا داب بن ارم بن حصرون بن فارص بن یہوذا بن یعقوب بن اسحٰق بن ابراہیم بن ابی الربیع نبی اللہ بن نبی اللہ علیہ السلام ایک عرصے سے مشہور چلا آتا ہے۔
+بعض کتابوں سے ظاہر ہوتا ہے کہ وہ یعنی سلیمانؑ د مشق گئے تھے۔ ابن ماکولا نے ان کا نسب نامہ جو بتایا ہے وہ وہی ہے جو ابن عساکر نے بتایا ہے۔
+اللہ تعالیٰ کا ارشاد ہے کہ ”ہم نے سلیمان کو داؤدؑ کا وارث بنایا“ اور یہ بھی فرمایا کہ ”اے لوگو! ہم نے اسے (سلیمانؑ کو) پرندوں کی منطق دی اور بہت سی چیزوں پر اسے اختیار دیا۔ اس پر یہ ہمارا ظاہر و با ہر فضل تھا“۔ جس کا مطلب یہ ہے کہ وہ پرندوں کی زبان سمجھتے اور دوسروں کو سمجھاتے تھے۔
```
غلطی: با ہر (اسپیس)
درست: باہر

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+حافظ ابن عساکر کے بقول سلیمانؑ کا پورا نام سلیمان بن داؤ دبن ایشا بن عويد بن عابر بن سلمون بن تحشون بن عمینا داب بن ارم بن حصرون بن فارص بن یہوذا بن یعقوب بن اسحٰق بن ابراہیم بن ابی الربیع نبی اللہ بن نبی اللہ علیہ السلام ایک عرصے سے مشہور چلا آتا ہے۔
+بعض کتابوں سے ظاہر ہوتا ہے کہ وہ یعنی سلیمانؑ د مشق گئے تھے۔ ابن ماکولا نے ان کا نسب نامہ جو بتایا ہے وہ وہی ہے جو ابن عساکر نے بتایا ہے۔
+اللہ تعالیٰ کا ارشاد ہے کہ ”ہم نے سلیمان کو داؤدؑ کا وارث بنایا“ اور یہ بھی فرمایا کہ ”اے لوگو! ہم نے اسے (سلیمانؑ کو) پرندوں کی منطق دی اور بہت سی چیزوں پر اسے اختیار دیا۔ اس پر یہ ہمارا ظاہر و با ہر فضل تھا“۔ جس کا مطلب یہ ہے کہ وہ پرندوں کی زبان سمجھتے اور دوسروں کو سمجھاتے تھے۔
+جہاں تک سلیمانؑ کو داؤد علیہ السلام کی وراثت ملنے کا تعلق ہے اس کا مطلب نبوت و بادشاہت ہے نہ کہ مال و زر جیسا کہ حدیث نبوی سے ظاہر ہے کہ انبیاء علیھم السلام کا ترکہ صدقہ ہوتا ہے جو ان کی اولاد کو نہیں ملتا بلکہ اس کے مستحق و ہ محتاج ہوتے ہیں جو ان کے رشتہ دار نہ ہوں کیونکہ انبیاء علیہ السلام کا فرضِ منصبی تبلیغ ِدین ہوتا ہے اس لیے وہ دنیا کے مال و زر سے بے نیاز ہوتے ہیں نہ اپنی اولاد کے لیے اسے ج ...
```
1۔ بلکہ اس کے مستحق و ہ محتاج ہوتے ہیں جو ان کے رشتہ دار نہ ہوں
 غلطی: و ہ (اسپیس)
درست: وہ

2۔ فرضِ منصبی تبلیغ ِدین ہوتا ہے اس لیے وہ دنیا کے
غلطی: تبلیغ ِدین (اسپیس)
درست: تبلیغِ دین

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+بعض کتابوں سے ظاہر ہوتا ہے کہ وہ یعنی سلیمانؑ د مشق گئے تھے۔ ابن ماکولا نے ان کا نسب نامہ جو بتایا ہے وہ وہی ہے جو ابن عساکر نے بتایا ہے۔
+اللہ تعالیٰ کا ارشاد ہے کہ ”ہم نے سلیمان کو داؤدؑ کا وارث بنایا“ اور یہ بھی فرمایا کہ ”اے لوگو! ہم نے اسے (سلیمانؑ کو) پرندوں کی منطق دی اور بہت سی چیزوں پر اسے اختیار دیا۔ اس پر یہ ہمارا ظاہر و با ہر فضل تھا“۔ جس کا مطلب یہ ہے کہ وہ پرندوں کی زبان سمجھتے اور دوسروں کو سمجھاتے تھے۔
+جہاں تک سلیمانؑ کو داؤد علیہ السلام کی وراثت ملنے کا تعلق ہے اس کا مطلب نبوت و بادشاہت ہے نہ کہ مال و زر جیسا کہ حدیث نبوی سے ظاہر ہے کہ انبیاء علیھم السلام کا ترکہ صدقہ ہوتا ہے جو ان کی اولاد کو نہیں ملتا بلکہ اس کے مستحق و ہ محتاج ہوتے ہیں جو ان کے رشتہ دار نہ ہوں کیونکہ انبیاء علیہ السلام کا فرضِ منصبی تبلیغ ِدین ہوتا ہے اس لیے وہ دنیا کے مال و زر سے بے نیاز ہوتے ہیں نہ اپنی اولاد کے لیے اسے ج ...
+یہ حدیث متعد د صحابہ رضی اللہ عنہ کے حوالے سے صحاح (صح ستہ) میں درج ہے۔
```
یہ حدیث متعد د صحابہ رضی اللہ عنہ کے حوالے 
غلطی: متعد د (اسپیس)
درست: متعدد

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+جہاں تک سلیمانؑ کو داؤد علیہ السلام کی وراثت ملنے کا تعلق ہے اس کا مطلب نبوت و بادشاہت ہے نہ کہ مال و زر جیسا کہ حدیث نبوی سے ظاہر ہے کہ انبیاء علیھم السلام کا ترکہ صدقہ ہوتا ہے جو ان کی اولاد کو نہیں ملتا بلکہ اس کے مستحق و ہ محتاج ہوتے ہیں جو ان کے رشتہ دار نہ ہوں کیونکہ انبیاء علیہ السلام کا فرضِ منصبی تبلیغ ِدین ہوتا ہے اس لیے وہ دنیا کے مال و زر سے بے نیاز ہوتے ہیں نہ اپنی اولاد کے لیے اسے ج ...
+یہ حدیث متعد د صحابہ رضی اللہ عنہ کے حوالے سے صحاح (صح ستہ) میں درج ہے۔
+حافظ ابو بکر بیہقی متعدد حوالوں سے بیان کرتے ہیں کہ سلیمان علیہ السلام نے ایک دن دمشق کے قریب درختوں کے ایک جھنڈ کے پاس سے گزرتے ہوئے وہاں دو چڑیوں کو دیکھا جو اپنی زبان میں کچھ باتیں کر رہے تھے انہوں نے اپنے ساتھیوں سے پوچھا: ”کیا تم جانتے ہو کہ یہ آپس میں کیا باتیں کر رہے ہیں؟“ اور ان کے انکار کرنے پر انہیں بتایا: ”یہ چڑیاں نر اور مادہ ہیں، نر مادہ سے کہہ رہا کہ ”اگر تو میری زوجیت میں آجائے تو  ...
+ بیہقی  کچھ دوسری اسناد کے حوالے سے کہتے ہیں کہ طیور کی بولی سمجھنے کے علاوہ سلیمانؑ دوسری تمام مخلوقات کی زبان سمجھنے کی خدا کے حکم سے قدرت رکھتے تھے اور انہیں ان پر اختیار حاصل تھا جس کا ثبوت ارشاد باری تعالىٰ (وَ أُوتِينَا مِنْ كُلِّ شَی ءِ) میں موجود ہے جس کا مطلب یہ ہے کہ وہ جملہ مخلوقات بشمول جنات سب کو حکم دے کر ان سے کام لے سکتے تھے۔
```
(وَ أُوتِينَا مِنْ كُلِّ شَی ءِ)
غلطی: شَی ءِ (اسپیس)
درست: شَیءِ

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+ثعلبی کہتے ہیں کہ بلقیس کی حکومت سے قبل ان کی قوم پر جو شخص حکمران تھا وہ شرابی اور عیش وعشرت کا دلدادہ تھا جس کی وجہ سے ساری قوم میں فسادات پھوٹ پڑے تھے اور سارے ملک میں انتشار پھیل گیا تھا۔ یہ دیکھ کر بلقیس نے اسے کچھ لوگوں کی مدد سے تہ تیغ کر کے اس کا سر اس کے قصر کے دروازے پر ٹنگوا دیا تھا اور اس حکمران سے نجات پانے کے بعد بلقیس کی ساری قوم پر اس کا سکہ بیٹھ گیا تھا اور وہ تمام کی تمام اس کے  ...
+ کہا جاتا ہے کہ اس نے اپنے شاہی محل میں جو تخت بنوایا تھا وہ زر و جواہر سے مزین ہونے کے علاوہ اپنی چھت کے لحاظ سے بے نظیر تھا کیونکہ اس میں سچ مچ کے ستارے لٹکتے نظر آتے تھے۔
+ملکہ سبا یعنی بلقیس کے اس کروفر کا حال سلیمانؑ سے جب بیان کیا گیا تو انہوں نے اسے اپنے پاس بلانے کا قصد کیا تو ایک جن نے ان سے عرض کیا کہ اگر ان کی اجازت ہو تو وہ بلقیس کو اس کے تخت سمیت ان کی خدمت میں لا کر حاضر کر دے روایت ہے کہ یہ پیشکش کرنے والے آصف بن برخیا تھے اور قوم جنات کے اس گروہ سے تعلق رکھتے تھے جو اللہ تعالیٰ پر ایمان لے آئی تھی۔ تا ہم سلیمانؑ نے پہلے طاہر ہدہد کو حکم دیا کہ وہ پہلے  ...
+چنانچہ سلیمانؑ کا یہ خط ہد ہد بلقیس کے پاس اسی طرح لے گیا جیسے پہلے کبوتر پیغامات اور خبریں لے جایا کرتے تھے۔ بلقیس نے سلیمانؑ کا وہ خط موصول ہونے کے بعد اپنے درباریوں سے مشورہ کیا اس قوم کی طرح سب کے سب سورج کی پرستش کرتے تھے اور بڑے متکبر و سرکش تھے اس لیے انہوں نے بلقیس کو مشورہ دیا کہ وہ ہرگز سلیمانؑ کے پاس نہ جائے لیکن ان کے اس مشورے پر جب بلقیس نے سلیمان علیہ السلام کی خدمت میں حاضر ہونے سے ...
```
اسے اس کے تحت سمیت ان کی خدمت میں حاضر کر دیا جائے۔
غلطی: تحت سمیت (املاء)
درست: تخت سمیت

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+ کہا جاتا ہے کہ اس نے اپنے شاہی محل میں جو تخت بنوایا تھا وہ زر و جواہر سے مزین ہونے کے علاوہ اپنی چھت کے لحاظ سے بے نظیر تھا کیونکہ اس میں سچ مچ کے ستارے لٹکتے نظر آتے تھے۔
+ملکہ سبا یعنی بلقیس کے اس کروفر کا حال سلیمانؑ سے جب بیان کیا گیا تو انہوں نے اسے اپنے پاس بلانے کا قصد کیا تو ایک جن نے ان سے عرض کیا کہ اگر ان کی اجازت ہو تو وہ بلقیس کو اس کے تخت سمیت ان کی خدمت میں لا کر حاضر کر دے روایت ہے کہ یہ پیشکش کرنے والے آصف بن برخیا تھے اور قوم جنات کے اس گروہ سے تعلق رکھتے تھے جو اللہ تعالیٰ پر ایمان لے آئی تھی۔ تا ہم سلیمانؑ نے پہلے طاہر ہدہد کو حکم دیا کہ وہ پہلے  ...
+چنانچہ سلیمانؑ کا یہ خط ہد ہد بلقیس کے پاس اسی طرح لے گیا جیسے پہلے کبوتر پیغامات اور خبریں لے جایا کرتے تھے۔ بلقیس نے سلیمانؑ کا وہ خط موصول ہونے کے بعد اپنے درباریوں سے مشورہ کیا اس قوم کی طرح سب کے سب سورج کی پرستش کرتے تھے اور بڑے متکبر و سرکش تھے اس لیے انہوں نے بلقیس کو مشورہ دیا کہ وہ ہرگز سلیمانؑ کے پاس نہ جائے لیکن ان کے اس مشورے پر جب بلقیس نے سلیمان علیہ السلام کی خدمت میں حاضر ہونے سے ...
+ اس کے بعد سلیمانؑ کی یہ طاقت نیز یہ دیکھ کر کہ تمام وحوش و طیور تک ان کے فرماں بردار ہیں ان کی شان و شوکت اوررعب و دبدبے کا اندازہ لگالیا اور ان کے دست حق پرست پر ایمان لے آئی۔
```
ان کی شان و شوکت اوررعب و دبدبے کا اندازہ لگالیا
1۔ غلطی: اوررعب (اسپیس)
درست: اور رعب

2۔ غلطی: لگالیا (اسپیس)
درست: لگا لیا

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+ملکہ سبا یعنی بلقیس کے اس کروفر کا حال سلیمانؑ سے جب بیان کیا گیا تو انہوں نے اسے اپنے پاس بلانے کا قصد کیا تو ایک جن نے ان سے عرض کیا کہ اگر ان کی اجازت ہو تو وہ بلقیس کو اس کے تخت سمیت ان کی خدمت میں لا کر حاضر کر دے روایت ہے کہ یہ پیشکش کرنے والے آصف بن برخیا تھے اور قوم جنات کے اس گروہ سے تعلق رکھتے تھے جو اللہ تعالیٰ پر ایمان لے آئی تھی۔ تا ہم سلیمانؑ نے پہلے طاہر ہدہد کو حکم دیا کہ وہ پہلے  ...
+چنانچہ سلیمانؑ کا یہ خط ہد ہد بلقیس کے پاس اسی طرح لے گیا جیسے پہلے کبوتر پیغامات اور خبریں لے جایا کرتے تھے۔ بلقیس نے سلیمانؑ کا وہ خط موصول ہونے کے بعد اپنے درباریوں سے مشورہ کیا اس قوم کی طرح سب کے سب سورج کی پرستش کرتے تھے اور بڑے متکبر و سرکش تھے اس لیے انہوں نے بلقیس کو مشورہ دیا کہ وہ ہرگز سلیمانؑ کے پاس نہ جائے لیکن ان کے اس مشورے پر جب بلقیس نے سلیمان علیہ السلام کی خدمت میں حاضر ہونے سے ...
+ اس کے بعد سلیمانؑ کی یہ طاقت نیز یہ دیکھ کر کہ تمام وحوش و طیور تک ان کے فرماں بردار ہیں ان کی شان و شوکت اوررعب و دبدبے کا اندازہ لگالیا اور ان کے دست حق پرست پر ایمان لے آئی۔
+ ثعلبی کہتے ہیں کہ سلیمانؑ اسے اپنی زوجیت میں لے آئے تھے اور اسے اس کی مملکت میں واپس کر دیا تھا بلکہ وہیں اس کے لیے تین بڑے شان دار محل عدنان ،سالحین اور بیتون بنوا دیئے تھے اور جب کبھی دوسرے شہروں سے ہوتے ہوئے یمن جاتے تو تین روز اس کے پاس ٹھہرا کرتے تھے جب کہ ابن اسحٰق بعض اہل ِعلم کے علاوہ وہب بن منبہ کے حوالے سے بیان کرتے ہیں کہ سلیمانؑ نے بلقیس سے شادی نہیں کی تھی بلکہ اس کی شادی ہمدان کے ب ...
```
ابن اسحٰق بعض اہل ِعلم کے علاوہ وہب بن منبہ کے حوالے
غلطی: اہل ِعلم (اسپیس)
درست: اہلِ علم

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+ اس کے بعد سلیمانؑ کی یہ طاقت نیز یہ دیکھ کر کہ تمام وحوش و طیور تک ان کے فرماں بردار ہیں ان کی شان و شوکت اوررعب و دبدبے کا اندازہ لگالیا اور ان کے دست حق پرست پر ایمان لے آئی۔
+ ثعلبی کہتے ہیں کہ سلیمانؑ اسے اپنی زوجیت میں لے آئے تھے اور اسے اس کی مملکت میں واپس کر دیا تھا بلکہ وہیں اس کے لیے تین بڑے شان دار محل عدنان ،سالحین اور بیتون بنوا دیئے تھے اور جب کبھی دوسرے شہروں سے ہوتے ہوئے یمن جاتے تو تین روز اس کے پاس ٹھہرا کرتے تھے جب کہ ابن اسحٰق بعض اہل ِعلم کے علاوہ وہب بن منبہ کے حوالے سے بیان کرتے ہیں کہ سلیمانؑ نے بلقیس سے شادی نہیں کی تھی بلکہ اس کی شادی ہمدان کے ب ...
+سورۂ ص میں اللہ تعالیٰ کا ارشاد ہے:
+”اور ہم نے داؤدؑ کو سلیمانؑ عطا کیے۔ بہت خوب بندے (تھے اور) وہ (خدا کی طرف) رجوع کرنے والے تھے۔ جب ان کے سامنے شام کو خاصے کے گھوڑے پیش کیے گئے۔ تو کہنے لگے کہ میں نے اپنے پروردگار کی یاد سے (غافل ہوکر)  مال کی محبت اختیار کی۔ یہاں تک کہ (آفتاب) پردے میں چھپ گیا۔ (بولے کہ) ان کو میرے پاس واپس لے آؤ پھر ان کی ٹانگوں اور گردنوں پر ہاتھ پھیرنے لگے۔ اور ہم نے سلیمانؑ کی آزمائش کی اور ان کے تخت پر ایک ...
```
اور اوروں کو بھی جو ز نجیروں میں جکڑے ہوئے تھے (ہم نے کہا) یہ ہماری بخشش ہے (چاہو) تو احسان کر دیا
1. غلطی: اور اوروں (اوروں کی جگہ دیودٔں)
درست: دیودٔں

2۔ غلطی: ز نجیروں (اسپیس)
درست: زنجیروں

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+”اور ہم نے داؤدؑ کو سلیمانؑ عطا کیے۔ بہت خوب بندے (تھے اور) وہ (خدا کی طرف) رجوع کرنے والے تھے۔ جب ان کے سامنے شام کو خاصے کے گھوڑے پیش کیے گئے۔ تو کہنے لگے کہ میں نے اپنے پروردگار کی یاد سے (غافل ہوکر)  مال کی محبت اختیار کی۔ یہاں تک کہ (آفتاب) پردے میں چھپ گیا۔ (بولے کہ) ان کو میرے پاس واپس لے آؤ پھر ان کی ٹانگوں اور گردنوں پر ہاتھ پھیرنے لگے۔ اور ہم نے سلیمانؑ کی آزمائش کی اور ان کے تخت پر ایک ...
+ان آیات شریفہ میں اللہ تعالیٰ نے داؤدؑ کو سلیمان علیہ السلام جیسا فرزند عطا فرمانے کے علاوہ داؤدؑ کے ان بیٹے یعنی سلیمانؑ کی کچھ صفات بھی بیان فرمائی ہیں۔ ایک تو یہ کہ وہ ہر حالت میں اللہ تعالیٰ کے تابع فرمان تھے اور ہر حال میں اس سے رجوع کیا کرتے تھے اور دوسرے یہ کہ وہ جانوروں کے ساتھ کس طرح شفقت سے پیش آتے تھے۔ اللہ تعالیٰ نے گھوڑوں کی ٹانگوں اور گردنوں پر جو ان کے ہاتھ پھیرنے کا ذکر فرمایا ان  ...
+ امام شافعیؒ نے بھی حضرت علی رضی اللہ عنہ کے اس قول کی تائید کی ہے اورمکحولؒ اور اوزاعیؒ نے اس بارے میں شدید جنگ کےمواقع کی شرط پیش کی ہے۔ ہم نے سورۂ نساء کی تفسیر کرتے ہوئے اس مسئلے پر بھی گفتگو کی ہے۔
+ابن جریر نے ان آیات میں ارشاد باری تعالیٰ کہ سلیمانؑ گھوڑوں کی ٹانگوں اور گردنوں پر ہاتھ پھیر اکرتے تھے کے بارے میں دو وضاحتی اقوال میں سے کہ سلیمانؑ نے ان کی ٹانگوں اور گردنوں سے پسینہ صاف کیا کرتے تھے یا تلوار سے ان کی ٹانگوں کے بال صاف کیا کرتے تھے اور ان کی گردنوں کے بال یا ان کے اجنہ (پر) کا ٹا کرتے تھے اوّل الذکر قول کو اختیار کیا ہے اور اس کی وجہ یہ بتائی ہے کہ دوسری وضاحت کے لحاظ سے یہ لا ...
```
1۔ گھوڑوں کی ٹانگوں اور گردنوں پر ہاتھ پھیر اکرتے تھے
غلطی: پھیر اکرتے (اسپیس)
درست: پھیرا کرتے

2۔ ان کی گردنوں کے بال یا ان کے اجنہ (پر) کا ٹا کرتے تھے
غلطی: اجنہ (پر) کا ٹا (اسپیس)
درست: اجنہ (پر) کاٹا

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+ امام شافعیؒ نے بھی حضرت علی رضی اللہ عنہ کے اس قول کی تائید کی ہے اورمکحولؒ اور اوزاعیؒ نے اس بارے میں شدید جنگ کےمواقع کی شرط پیش کی ہے۔ ہم نے سورۂ نساء کی تفسیر کرتے ہوئے اس مسئلے پر بھی گفتگو کی ہے۔
+ابن جریر نے ان آیات میں ارشاد باری تعالیٰ کہ سلیمانؑ گھوڑوں کی ٹانگوں اور گردنوں پر ہاتھ پھیر اکرتے تھے کے بارے میں دو وضاحتی اقوال میں سے کہ سلیمانؑ نے ان کی ٹانگوں اور گردنوں سے پسینہ صاف کیا کرتے تھے یا تلوار سے ان کی ٹانگوں کے بال صاف کیا کرتے تھے اور ان کی گردنوں کے بال یا ان کے اجنہ (پر) کا ٹا کرتے تھے اوّل الذکر قول کو اختیار کیا ہے اور اس کی وجہ یہ بتائی ہے کہ دوسری وضاحت کے لحاظ سے یہ لا ...
+ سلیمانؑ کے گھوڑوں کی تعداد بعض روایات میں بیسں ہزار اور بعض میں دس ہزار بتائی گئی ہے اور یہ بھی بتایا گیا ہے کہ ان میں سے بیس گھوڑے پر دار تھے۔
+ابو داؤدؒ نے اپنی کتاب سنن میں دوسرے متعدد راویوں کے حوالے سے حضرت عائشہ رضی اللہ عنہ کی زبانی یہ حدیث بیان کی ہے کہ غزوۂ تبوک کے سفر میں گرمی کی وجہ سے انہوں نے اپنی سواری کا پر دہ ایک طرف سے ذرا سا کھول رکھا تھا کہ آنحضرت ﷺنے ان کی سواری کے قریب آکر اس کی وجہ دریافت فرمائی۔ حضرت عائشہ رضی اللہ عنہ نے جواب دیا: ”میں گھوڑے دیکھ رہی تھی“۔ آپؐ نے فرمایا:”کیوں؟ کیا آپ نے اس سے قبل کبھی گھوڑے نہیں د ...
```
1۔ دوسرے متعدد راویوں کے حوالے سے حضرت عائشہ رضی اللہ عنہ کی زبانی یہ حدیث بیان
غلطی: رضی اللہ عنہ (املاء)
درست: رضی اللہ عنہا
یہ غلطی پوری پیراگراف میں دیکھی گئی ہے تو جہاں بھی رضی اللہ عنہ لکھا گیا ہے اسکو رضی اللہ عنہا میں change کر کے لکھیں

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+ابن جریر نے ان آیات میں ارشاد باری تعالیٰ کہ سلیمانؑ گھوڑوں کی ٹانگوں اور گردنوں پر ہاتھ پھیر اکرتے تھے کے بارے میں دو وضاحتی اقوال میں سے کہ سلیمانؑ نے ان کی ٹانگوں اور گردنوں سے پسینہ صاف کیا کرتے تھے یا تلوار سے ان کی ٹانگوں کے بال صاف کیا کرتے تھے اور ان کی گردنوں کے بال یا ان کے اجنہ (پر) کا ٹا کرتے تھے اوّل الذکر قول کو اختیار کیا ہے اور اس کی وجہ یہ بتائی ہے کہ دوسری وضاحت کے لحاظ سے یہ لا ...
+ سلیمانؑ کے گھوڑوں کی تعداد بعض روایات میں بیسں ہزار اور بعض میں دس ہزار بتائی گئی ہے اور یہ بھی بتایا گیا ہے کہ ان میں سے بیس گھوڑے پر دار تھے۔
+ابو داؤدؒ نے اپنی کتاب سنن میں دوسرے متعدد راویوں کے حوالے سے حضرت عائشہ رضی اللہ عنہ کی زبانی یہ حدیث بیان کی ہے کہ غزوۂ تبوک کے سفر میں گرمی کی وجہ سے انہوں نے اپنی سواری کا پر دہ ایک طرف سے ذرا سا کھول رکھا تھا کہ آنحضرت ﷺنے ان کی سواری کے قریب آکر اس کی وجہ دریافت فرمائی۔ حضرت عائشہ رضی اللہ عنہ نے جواب دیا: ”میں گھوڑے دیکھ رہی تھی“۔ آپؐ نے فرمایا:”کیوں؟ کیا آپ نے اس سے قبل کبھی گھوڑے نہیں د ...
+حضرت عائشہ رضی اللہ عنہ فرماتی ہیں کہ ان سے یہ سن کر آنحضرت صلی اللہ صلى الله عليہ وسلم نے فرمایا: ”پر دار گھوڑے! کیا کہیں گھوڑے بھی پردار ہوتے ہیں؟“ آپؐ کا یہ سوال سن کر حضرت عائشہ رضی اللہ عنہ بولیں: ”میں نے سنا ہے کہ سلیمانؑ کے گھوڑوں میں کچھ گھوڑے پر داربھی ہوتے تھے“۔
```
1۔ حضرت عائشہ رضی اللہ عنہ فرماتی ہیں کہ ان سے
غلطی: رضی اللہ عنہ (املاء)
درست: رضی اللہ عنہا

2۔ آنحضرت صلی اللہ صلى الله عليہ وسلم نے فرمایا: ”پر دار گھوڑے! کیا
غلطی: صلی اللہ صلى الله عليہ وسلم (املاء)
درست: صلی اللہ علیہ وسلم


3۔ کچھ گھوڑے پر داربھی ہوتے تھے“۔
غلطی: پر داربھی (اسپیس)
درست: پردار بھی

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+ سلیمانؑ کے گھوڑوں کی تعداد بعض روایات میں بیسں ہزار اور بعض میں دس ہزار بتائی گئی ہے اور یہ بھی بتایا گیا ہے کہ ان میں سے بیس گھوڑے پر دار تھے۔
+ابو داؤدؒ نے اپنی کتاب سنن میں دوسرے متعدد راویوں کے حوالے سے حضرت عائشہ رضی اللہ عنہ کی زبانی یہ حدیث بیان کی ہے کہ غزوۂ تبوک کے سفر میں گرمی کی وجہ سے انہوں نے اپنی سواری کا پر دہ ایک طرف سے ذرا سا کھول رکھا تھا کہ آنحضرت ﷺنے ان کی سواری کے قریب آکر اس کی وجہ دریافت فرمائی۔ حضرت عائشہ رضی اللہ عنہ نے جواب دیا: ”میں گھوڑے دیکھ رہی تھی“۔ آپؐ نے فرمایا:”کیوں؟ کیا آپ نے اس سے قبل کبھی گھوڑے نہیں د ...
+حضرت عائشہ رضی اللہ عنہ فرماتی ہیں کہ ان سے یہ سن کر آنحضرت صلی اللہ صلى الله عليہ وسلم نے فرمایا: ”پر دار گھوڑے! کیا کہیں گھوڑے بھی پردار ہوتے ہیں؟“ آپؐ کا یہ سوال سن کر حضرت عائشہ رضی اللہ عنہ بولیں: ”میں نے سنا ہے کہ سلیمانؑ کے گھوڑوں میں کچھ گھوڑے پر داربھی ہوتے تھے“۔
+حضرت عائشہ رضی اللہ عنہ فرماتی ہیں کہ ان کا یہ جواب سن کر آنحضرتؐ ہنس پڑے جس سے آپ کے دندان مبارک نظرآنے لگے۔
```
حضرت عائشہ رضی اللہ عنہ فرماتی ہیں کہ ان
غلطی: رضی اللہ عنہ (املاء)
درست: رضی اللہ عنہا

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+حضرت عائشہ رضی اللہ عنہ فرماتی ہیں کہ ان سے یہ سن کر آنحضرت صلی اللہ صلى الله عليہ وسلم نے فرمایا: ”پر دار گھوڑے! کیا کہیں گھوڑے بھی پردار ہوتے ہیں؟“ آپؐ کا یہ سوال سن کر حضرت عائشہ رضی اللہ عنہ بولیں: ”میں نے سنا ہے کہ سلیمانؑ کے گھوڑوں میں کچھ گھوڑے پر داربھی ہوتے تھے“۔
+حضرت عائشہ رضی اللہ عنہ فرماتی ہیں کہ ان کا یہ جواب سن کر آنحضرتؐ ہنس پڑے جس سے آپ کے دندان مبارک نظرآنے لگے۔
+بعض علماء کا بیان ہے کہ جب کچھ لوگ اللہ کے نام پر اپنے چوپائے چھوڑ دیتے ہیں تو اللہ تعالیٰ ان کے بدلے میں انہیں کوئی ایسی چیز عطا فرما دیتا ہے جو ان کے لیے ان سے زیادہ بہتر ثابت ہوتی ہے جب کہ وہ چوپائے جو اس کے نام پر چھوڑے جاتے ہیں وہ آزادی سے کبھی کہیں پھرتے رہتے ہیں تا کہ ان کی تعداد زیادہ ہوتی ہے کہیں کم۔
+اس مسئلے پر ہم ان شاء اللہ آگے چل کر تفصیل سے گفتگو کریں گے۔ ویسے امام احمدؒ فرماتے ہیں کہ ان سے اسماعیل اور سلیمان بن مغیرہ نے حمید بن ہلال، ابی قتادہ اور ابی الدھما کے حوالے سے کہ آخر الذکر دونوں اکثر سفر کیا کرتے تھے بیان کیا کہ ان کے پاس دورانِ سفر میں ایک بدو آیا اور ان سے کہا کہ رسول اللہﷺ نے ایک روز اس کا ہاتھ اپنے دست مبارک میں لے کر فرمایا کہ جب کوئی شخص کوئی چیز خدا کی راہ میں فدیہ کر د ...
```
اس نے تمہیں پہلے ہی عطا فرمار کھی ہے تمہاری بھلائی سمجھتا ہو۔
غلطی: فرمار کھی (اسپیس issue)
درست: فرما رکھی

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+السدی مذکورہ بالا حوالوں سے مزید بیان کرتے ہیں کہ چونکہ اللہ تعالیٰ کو یقیناََ یہ پسند نہ تھا کہ سلیمانؑ اپنی آنکھوں کے سامنے بیت المقدس کی تباہی دیکھیں اس لیے اس نے اس سے قبل ان کی موت کا حکم دے دیا۔ چنانچہ جب وہ نماز کے لیے محراب میں تشریف لے گئے تو اللہ تعالیٰ کے حکم سے وہیں وفات پاگئے لیکن وہ اس وقت نماز سے فارغ ہو کر اپنا عصا لیے غالباً محراب سے باہر آنے کا قصد کر رہے تھے کہ ان کی روح قبض کر ...
+بہر کیف سلیمانؑ کی وفات کے کافی عرصہ بعد جیسا کہ اس روایت میں مزید بیان کیا گیا ہے، جب جنات کو اس کی خبر ہوئی تو وہ آپس میں کہنے لگے کہ اگر انہیں ان کی وفات کی قبل از وقت اطلاع ہو جاتی تو وہ انہیں بہترین کھانے اور بہتر سے بہتر مشروبات پیش کرتے لیکن مندرجہ بالا آیت قرآنی سے ثابت ہوتا ہے کہ ان کی وفات کی خبر پانے کے بعد جنات نے آپس میں کہاتھا کہ اگر انہیں ان کی وفات کی خبر پہلے ہو جاتی تو وہ ان کے  ...
+ابن مسعود متعلقہ آیات قرآنی کی توضیح کرتے ہوئے بیان کرتے ہیں کہ سلیمانؑ کی وفات اور لوگوں میں ان کی شہرت کا درمیانی فصل پورے ایک سال پر محیط تھا، جب کہ جنات اس دوران میں انہیں زندہ سمجھتے رہے تھے کیونکہ اس دوران میں جیسا کہ بعض روایات سے ظاہر ہوتا ہے، محراب مسجد اور ان کے درمیان آگ کا ایک الاؤ انہیں نظر آتا رہا جس کے خوف سے انہوں نے محراب کے قریب جانے کی جرأت نہیں کی اور انسانوں سے یہی کہتے رہے  ...
+ بہر کیف جب لوگوں کو سلیمانؑ کی وفات کی خبر ہوئی تو انہوں نے جنات کے پہلے بیان کو کذب پرمحمول کیا اور بعض نے یہ بھی کہا کہ جنات کو اس کا علم کس طرح ہو سکتا تھا جب کہ سلیمانؑ کے عصا کو دیمک کا کیڑا رات دن ایک سال تک برابر کھاتا رہا۔ جس کے اختتام پر ان کا جسد خاکی زمین پر گرا تھا۔
```
غلطی: کذب پرمحمول (اسپیس)
درست: کذب پر محمول

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+بہر کیف سلیمانؑ کی وفات کے کافی عرصہ بعد جیسا کہ اس روایت میں مزید بیان کیا گیا ہے، جب جنات کو اس کی خبر ہوئی تو وہ آپس میں کہنے لگے کہ اگر انہیں ان کی وفات کی قبل از وقت اطلاع ہو جاتی تو وہ انہیں بہترین کھانے اور بہتر سے بہتر مشروبات پیش کرتے لیکن مندرجہ بالا آیت قرآنی سے ثابت ہوتا ہے کہ ان کی وفات کی خبر پانے کے بعد جنات نے آپس میں کہاتھا کہ اگر انہیں ان کی وفات کی خبر پہلے ہو جاتی تو وہ ان کے  ...
+ابن مسعود متعلقہ آیات قرآنی کی توضیح کرتے ہوئے بیان کرتے ہیں کہ سلیمانؑ کی وفات اور لوگوں میں ان کی شہرت کا درمیانی فصل پورے ایک سال پر محیط تھا، جب کہ جنات اس دوران میں انہیں زندہ سمجھتے رہے تھے کیونکہ اس دوران میں جیسا کہ بعض روایات سے ظاہر ہوتا ہے، محراب مسجد اور ان کے درمیان آگ کا ایک الاؤ انہیں نظر آتا رہا جس کے خوف سے انہوں نے محراب کے قریب جانے کی جرأت نہیں کی اور انسانوں سے یہی کہتے رہے  ...
+ بہر کیف جب لوگوں کو سلیمانؑ کی وفات کی خبر ہوئی تو انہوں نے جنات کے پہلے بیان کو کذب پرمحمول کیا اور بعض نے یہ بھی کہا کہ جنات کو اس کا علم کس طرح ہو سکتا تھا جب کہ سلیمانؑ کے عصا کو دیمک کا کیڑا رات دن ایک سال تک برابر کھاتا رہا۔ جس کے اختتام پر ان کا جسد خاکی زمین پر گرا تھا۔
+سلیمانؑ کی وفات کی خبر مشہور ہونے کے بعد جب بیت المقدس میں داخل ہو کر لوگوں نے ان کا عصا دیکھا تو وہ کھوکھلا ہو کرقریب قریب خاک ہو چکا تھا البتہ اس کے اوپر ایسی مٹی کے کچھ آثار بھی تھے جو جھاڑیوں کی جڑوں میں ہوتی ہے۔
```
تو وہ کھوکھلا ہو کرقریب قریب خاک ہو چکا تھا
غلطی: ہو کرقریب (اسپیس)
درست: ہو کر قریب

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_1.md:
```diff
@@ -0,0 +1,246 @@
+سلیمانؑ کی وفات کی خبر مشہور ہونے کے بعد جب بیت المقدس میں داخل ہو کر لوگوں نے ان کا عصا دیکھا تو وہ کھوکھلا ہو کرقریب قریب خاک ہو چکا تھا البتہ اس کے اوپر ایسی مٹی کے کچھ آثار بھی تھے جو جھاڑیوں کی جڑوں میں ہوتی ہے۔
+یہ روایت اسرائیلیات سے ماخوذ ہے لیکن اس کی صحت و تکذیب یقینی طور پر بیان نہیں کی جاسکتی۔
+
+ ابوداؤ داپنی کتاب القدر میں بیان کرتے ہیں کہ سلیمانؑ نے ملک الموت سے کہا تھا کہ جب اسے ان کی روح قبض کرنے کا حکم دیا جائے تو وہ انہیں اس کی اطلاع دے دے لیکن ملک الموت نے انہیں جواب دیا تھا کہ ہرنفس کی موت کا وقت تو مقرر ہے لیکن خود اسے اس کی خبر نہیں ہوتی کیونکہ عین وقت پر ا سے مرنے والے کا نام بتا کر اس کی روح قبض کرنے کا اللہ تعالیٰ کی طرف سے حکم دیا جاتا ہے۔
```
1۔ ابو داؤ داپنی کتاب القدر میں بیان کرتے ہیں کہ سلیمانؑ نے
غلطی: ابو داؤ داپنی (اسپیس)
درست: ابو داؤد اپنی

2۔ کیونکہ عین وقت پر ا سے مرنے والے کا
غلطی: ا سے (اسپیس)
درست: اسے

Laaibah reviewed (changes requested):
I have thoroughly reviewed your first chapter today. While it shows improvement, there are still some errors that I have highlighted for your ease of correction. I hope that these mistakes will not persist in the upcoming chapters. I look forward to seeing more accuracy in your work.

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_5.md:
```diff
@@ -0,0 +1,59 @@
+
+## حجازی عربوں کے بنی اسماعیل کا ذکر جو زمانہ جاہلیت سے قبل زمانہ بعثتِ نبوی ﷺ تک وہاں کے حکمران رہے
+
+حضرت اسماعیلؑ کا ذکر اس سے قبل دوسرے انبیاء علیہ السلام کے ذکر کے ساتھ کیا جاچکا ہے اور یہ بھی بیان کیا جا چکا ہے کہ جب ان کے والد گرامی حضرت ابراہیمؑ  انہیں جو اس وقت شیر خوار تھے اور ان کی والدہ ماجدہ حضرت ہاجر  کو خدا کے حکم کے مطابق فاران کے ویران پہاڑوں کے درمیان تنہا چھوڑ کر چلے گئے تھے نیز یہ کہ اس وقت ان کی والدہ کے پاس کھانے کے لیے ایک جراب میں تھوڑی سی املی اور ایک چھوٹی سی کپی میں تھو ...
```
ان کی والدہ ماجدہ حضرت ہاجر  کو خدا کے حکم کے
غلطی: ہاجر  (املاء)
درست: ہاجرہ رضی اللہ عنہا

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_5.md:
```diff
@@ -0,0 +1,59 @@
+
+حضرت اسماعیلؑ کا ذکر اس سے قبل دوسرے انبیاء علیہ السلام کے ذکر کے ساتھ کیا جاچکا ہے اور یہ بھی بیان کیا جا چکا ہے کہ جب ان کے والد گرامی حضرت ابراہیمؑ  انہیں جو اس وقت شیر خوار تھے اور ان کی والدہ ماجدہ حضرت ہاجر  کو خدا کے حکم کے مطابق فاران کے ویران پہاڑوں کے درمیان تنہا چھوڑ کر چلے گئے تھے نیز یہ کہ اس وقت ان کی والدہ کے پاس کھانے کے لیے ایک جراب میں تھوڑی سی املی اور ایک چھوٹی سی کپی میں تھو ...
+
+یہ واقعہ ابن عباس کی بیان کردہ تفصیلی روایت میں ہے جسے بخاریؒ نے بھی بیان کیا ہے اور ان دونوں کے حوالےسے پہلے تحریر کیا جا چکا ہے۔
```
یہ واقعہ ابن عباس کی بیان کردہ تفصیلی روایت میں ہے جسے بخاریؒ نے بھی بیان کیا ہے اور ان دونوں کے حوالےسے پہلے تحریر کیا جا چکا ہے۔
1. غلطی: عباس (املاء)
درست: عباسؓ

2.غلطی: حوالےسے (اسپیس)
درست: حوالے سے

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_5.md:
```diff
@@ -0,0 +1,59 @@
+
+یہ واقعہ ابن عباس کی بیان کردہ تفصیلی روایت میں ہے جسے بخاریؒ نے بھی بیان کیا ہے اور ان دونوں کے حوالےسے پہلے تحریر کیا جا چکا ہے۔
+
+حضرت اسماعیل علیہ السلام اور ان کی والدہ حضرت ہاجرہ کے کچھ دن قیام کے بعد وہاں قدیم ترین عربی قبیلے جرہم کی ایک جماعت پانی کی تلاش میں آئی جسے حضرت ہاجرہ نے اللہ تعالیٰ کا اپنی رحمت سے انہیں عطا فرمودہ اس کنوئیں کا پتہ بتایا جس کے بعد وہ لوگ بھی ہمیشہ کے لیے وہیں آباد ہو گئے۔
```
حضرت اسماعیل علیہ السلام اور ان کی والدہ حضرت ہاجرہ کے کچھ
غلطی: ہاجرہ (املاء)
درست: ہاجرہؓ

حضرت ہاجرہ نے اللہ تعالیٰ کا اپنی رحمت سے انہیں
غلطی: ہاجرہ (املاء)
درست: ہاجرہؓ

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_5.md:
```diff
@@ -0,0 +1,59 @@
+
+اس مشہور و معروف واقعے کے بعد حضرت اسماعیلؑ نے قبیلہ جرہم کی ایک لڑکی سے شادی کر لی تھی لیکن اسے طلاق دے کر پھر اسی قبیلے کی جس دوسری لڑکی سے انہوں نے شادی کی اس کا نام سیدہ بنت مضاض بن عمر و جرہمی تھا۔
+
+سیدہ بنت مضاض سے حضرت اسماعیلؑ کے بارہ بیٹے پیدا ہوئے جن کا پہلے ذکر کیا جا چکا ہے۔ ان کے نام نابت، قیدر، منشا، سمع،  و د ما، مادر، یوکر، نیشی،  وطیما اور قیدما تھے۔
```
ان کے نام نابت، قیدر، منشا، سمع، و د ما،
غلطی: و د ما (اسپیس)
درست: ودما

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_5.md:
```diff
@@ -0,0 +1,59 @@
+
+سیدہ بنت مضاض سے حضرت اسماعیلؑ کے بارہ بیٹے پیدا ہوئے جن کا پہلے ذکر کیا جا چکا ہے۔ ان کے نام نابت، قیدر، منشا، سمع،  و د ما، مادر، یوکر، نیشی،  وطیما اور قیدما تھے۔
+
+یہ نام محمد ابن اسحاق نے اہل کتاب کی روایات کے حوالے سے بتائے ہیں جن میں ابن جریر اور طبری نے کچھ تبدیلیاں کی ہیں۔ بہر حال یہ بات متفق علیہ ہے کہ حضرت اسماعیلؑ کی بیٹی صرف ایک تھی جس کا نام نسمہ تھا اور انہوں نے اس کی شادی اپنے بھائی حضرت اسقؑات کے بیٹے عیصو سے کی تھی اور اس کے بطن سے روم اور فارس پیدا ہوئے تھے جب کہ دو روایتوں میں سے ایک روایت کے مطابق اشبان بھی نسمہ ہی کے بطن سے پیدا ہوئے تھے۔
```
اس کی شادی اپنے بھائی حضرت اسقؑات کے بیٹے عیصو سے کی تھی اور اس کے بطن
غلطی: حضرت اسقؑات (املاء)
درست: حضرت اسحاقؑ

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_5.md:
```diff
@@ -0,0 +1,59 @@
+
+یہ نام محمد ابن اسحاق نے اہل کتاب کی روایات کے حوالے سے بتائے ہیں جن میں ابن جریر اور طبری نے کچھ تبدیلیاں کی ہیں۔ بہر حال یہ بات متفق علیہ ہے کہ حضرت اسماعیلؑ کی بیٹی صرف ایک تھی جس کا نام نسمہ تھا اور انہوں نے اس کی شادی اپنے بھائی حضرت اسقؑات کے بیٹے عیصو سے کی تھی اور اس کے بطن سے روم اور فارس پیدا ہوئے تھے جب کہ دو روایتوں میں سے ایک روایت کے مطابق اشبان بھی نسمہ ہی کے بطن سے پیدا ہوئے تھے۔
+
+بہر کیف بنی جرہم ہی مکے میں کثرت سے آباد ہوتے چلے گئے اور ان کا حکمران کےی  یا بیت اللہ کا متولی کےی  اسی قبیلے کی دو لڑکیوں میں سے ایک کے بطن سے پیدا ہونے والے نابت بن اسماعیل کو تسلیم کیا گیا تھا اور انہی کو زمزم کا نگران بھی بنایا گیا تھا۔
```
ان کا حکمران کےی یا بیت اللہ کا متولی کےی اسی قبیلے
غلطی: کےی (املاء)
درست: کہیے

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_5.md:
```diff
@@ -0,0 +1,59 @@
+
+بہر حال بنی خزاعہ ہی کے بارے میں کہا جاتا ہے کہ ان کے دو اشخاص عمرو اور حارث نے بنی جرہم کو سرداری سے معطل کر کے بیت اللہ کی تولیت اور زمزم کی نگرانی اپنے تصرف میں کر لی۔ اور حجر اسود کے علاوہ جو پہلے ہی سے بیت اللہ میں موجود تھا دوسرے لا تعداد پتھر وہاں لالا کر جمع کر دیئے جس کے بعد وہ یمن گئے اور وہاں سے بہت سا سونا لا کر چاہ زمزم کی تہ میں دفن کر دیا۔
+
+اس کے بارے میں عمرو بن حارث نے بہت سے فخریہ اشعار بھی کہے ہیں جنہیں اکثر مؤرخین نے اپنی کتابوں میں پیش کیا ہے۔ ابن اسحٰق نے عمرو بن حارث بن مضاض کے ساتھ بنی بکر اور غیشان کا بھی ذکر کیا ہے جو اس کے بعد مکے آکر آبادہوئے تھے۔
```
جو اس کے بعد مکے آکر آبادہوئے تھے۔

غلطی: آبادہوئے (اسپیس)
درست: آباد ہوئے

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_5.md:
```diff
@@ -0,0 +1,59 @@
+
+اس کے بارے میں عمرو بن حارث نے بہت سے فخریہ اشعار بھی کہے ہیں جنہیں اکثر مؤرخین نے اپنی کتابوں میں پیش کیا ہے۔ ابن اسحٰق نے عمرو بن حارث بن مضاض کے ساتھ بنی بکر اور غیشان کا بھی ذکر کیا ہے جو اس کے بعد مکے آکر آبادہوئے تھے۔
+
+ابن ہشام کہتے ہیں کہ بنی بکر اور غیشان نے مکے آکر جو اشعار کہے تھے اور متعددعربی ادب کے مجموعہ ہائے کلام میں ملتے ہیں انہیں عربی شاعری کی ابتدا کہا جاتا ہے۔
```
جو اشعار کہے تھے اور متعددعربی ادب کے مجموعہ
غلطی: متعددعربی (اسپیس)
درست: متعدد عربی

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_5.md:
```diff
@@ -0,0 +1,59 @@
+
+ابن اسحٰق کہتے ہیں کہ عمرو بن حارث کے بعد خزاعہ ہی کے قبیلے سے ایک شخص کو جس کا نام غبشان یاغیشان تھا۔ بیت اللہ کی تولیت سپرد کی گئی جن میں عمرو بن حارث، قریش اور بنی کنانہ کے وہ لوگ بھی شامل ہو گئے جو ان قوموں میں شامل ہو گئے تھے۔
+
+خانہ کعبہ کی تولیت یکے بعد دیگرے بنی خزاعہ ہی میں منتقل ہوتی رہی حتٰی کہ وہ ان کے آخری شخص حلیل بن جشیہ بن سلول ابن کعب بن عمرو بن ربیعہ خزاعی تک پہنچی جس نے قصی بن کلاب کی بیٹی حبی سے شادی کی اور اس کے بطن سے اس کے چار بیٹے عبدالدار، عبد مناف، عبد العزی اور عبدا پیدا ہوئے اور پھر بیت اللہ کی تولیت بھی یکے بعد دیگرے انہی میں منتقل ہوتی رہی جس کا تفصیلی ذکر ہم حسب محل ان شاء اللہ آگے چل کر کریں گے ...
```
بہر کیف کےی کی تولیت بنی خزاعہ میں قریباً تین سو سال اور بعض روایات کے مطابق پانچ سو سال تک چلی۔

غلطی: کےی (املاء)
درست: کعبے

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_5.md:
```diff
@@ -0,0 +1,59 @@
+
+بنی خزاعہ میں جس شخص کا نام سوس تھا اس کے بارے میں کہا گیا ہے کہ خانہ کعبہ کی تولیت کے زمانے میں اس کے پاس بیس ہزار اونٹ تھے جو اس تعداد میں مکے کے کسی اور تنہا شخص کے پاس نہیں تھے اور اسی لیے مکہ پر اس کی سرداری مسلم تھی۔ اس کے علاوہ وہ اپنے ان اونٹوں میں سے ہر سال کعبہ کی زیارت اور حج کے لیے آنے والے عربی قبائل کی ضیافت کے لیے کثیر تعداد میں اونٹ ذبح کرایا کرتا تھا اور مکے آنے والوں کو زر نقد ب ...
+
+ابن ہشام کہتے ہیں کہ ان سے بعض اہل علم نے بیان کیا کہ مکے کا ایک شخص عمرو بن لحی اپنے کسی کام سے شام گیا تو راستے میں بلقاء میں ایک آدھ روز کے لیے ٹھہرا جہاں عمالیق یعنی قبیلہ عا لیق کے لوگ رہتے تھے۔ عمرو بن لحی نے دیکھا کہ وہ لوگ بت پرست ہیں اور خاص بتوں کی پرستش کرتے ہیں اور انہوں نے ان بتوں کے الگ الگ نام بھی رکھ چھوڑے ہیں۔ یہ دیکھ کر اس نے ان لوگوں سے پوچھا کہ یہ کیسی عبادت ہے جو وہ کرتے ہیں  ...
```
ایک آدھ روز کے لیے ٹھہرا جہاں عمالیق یعنی قبیلہ عا لیک کے لوگ رہتے تھے۔
غلطی: عا لیک (املاء)
درست: عمالیق

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_5.md:
```diff
@@ -0,0 +1,59 @@
+
+******
+
+عمرو بن لحی کے اس سوال کے جواب میں وہ لوگ بولے کہ وہ ان بتوں سے خشک سالی کے زمانے میں بارش کے لیے مدد مانگتے ہیں تو بارش ہو جاتی ہے۔ اس کے علاوہ جب بھی انہیں کوئی مشکل پیش آتی ہے اس وقت بھی وہ انہیں بتوں سے امداد طلب کرتے ہیں اور وہی بت ان کے حسبِ خواہش ان کی مدد کرتے ہیں۔
```
اس وقت بھی وہ انہیں بتوں سے امداد طلب
غلطی: انہیں بتوں (اسپیس)
درست: انہی بتوں

Laaibah commented on books/al_bidaya_wa_nihaya_vol_2/content/chap_5.md:
```diff
@@ -0,0 +1,59 @@
+
+ان عمالیق نے عمرو بن لحی کو اپنے ان بتوں میں سے جن کی وہ پوجا کرتے تھے ایک بت اس کی درخواست پر اسے دے دیا جسے لے کر وہ مکے آیا اور ان عمالیق کے عقائد اہل مکہ سے کچھ انداز میں بیان کیے اور ان بتوں کی کرشمہ سازی کا بھی ذکر کیا تو انہوں نے اسے اس بت کو بیت اللہ میں رکھنے کی اجازت دے دی بلکہ خود بھی اس کی دیکھا دیکھی اس کی پرستش کرنے لگے۔ بعض روایات میں اس پہلے بت کا نام جو بیت اللہ میں رکھا گیا ہبل  ...
+
+کہا جاتا ہے کہ جب تک بنی اسماعیل مکے تک محدود رہے ان میں بت پرستی کا نام و نشان تک نہ تھا لیکن جوں جوں وہ عرب کے دوسرے علاقوں میں آباد ہوتے گئے اور وہ جب بھی کسی جگہ سے مکے آتے یا کوئی اور شخص ان کے ساتھ آتا تو وہ اپنے ساتھ کوئی نہ کوئی بت ضرور لاتا اور پھر اسے بیت اللہ میں سجا دیا جاتا جہاں لوگ اس کی پرستش کرنے لگتے ۔
```
لوگ اس کی پرستش کرنے لگتے ۔

غلطی: لگتے ۔ (اسپیس)
درست: لگتے۔

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

chpa #5 correct all errors & pushed.

Sajidbhatti526 committed:
feature/#11: Abidaya wa Nihaya Vol 2

chap #1 correct all errors & pushed again.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

All chaps k paragraphs correct kiye hain.

Sajidbhatti526 committed:
feature/#11: Albidaya wa NIhaya Vol 2

chap 2 PR. and pushed

Sajidbhatti526 committed:
feature/#11: Abidaya wa NIhaya Vol 2

chap #2. PR & PUSH

Sajidbhatti526 committed:
feature/#11: Albidaya wa NIhaya Vol 2

Chap # 3, qissa yajoj majoj tk PR kr k push kr diya h.

Sajidbhatti526 committed:
feature/#11: Albiday wa NIhaya Vol 2

chap 3 & 4, PR  kr k push kr diye hain.

Sajidbhatti526 committed:
feature/#11: Albidaya wa NIhaya Vol 2

All work jo typed h 100% PR kr k correct ho gya h or push b kr diya h.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

Chap # 6. k kuch mazeed topix pushed.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

Chap # 6, k mazeed topix pushed.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

Chap # 6. ka aik topic or push kr diay h. in ka abi STRICT PR krna h maine.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

some changes in Chap # 6.

Sajidbhatti526 committed:
feature/#11: Albidaya wa NIhaya Vol 2

push 1 more topic of Chap # 6.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

some changes in chap 6

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihyay Vol 2

add some change.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

add 1 more topic in chap 6

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

add footnote in last topic of chap 6.

Sajidbhatti526 committed:
feature/#11: Albidaya wa Nihaya Vol 2

chap #7. complete and pushed.

Sajidbhatti526 committed:
feaure/#11: Albidaya wa Nihaya Vol 2

chap # 6. complete and pushed.

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
apdate

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
Merge branch 'feature/11-albidaya-wa-nihaya-vol-2' of https://github.com/segullshairbutt/islamopedia-books into feature/11-albidaya-wa-nihaya-vol-2

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

segullshairbutt committed:
Merge branch 'main' into feature/11-albidaya-wa-nihaya-vol-2

segullshairbutt committed:
feature/#11-albidaya-wa-nihaya-vol-2

The book is properly formatted.

muhammadnuman383 committed:
UPDATE

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
UPDATE

muhammadnuman383 committed:
UPDATE

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

segullshairbutt committed:
Merge branch 'main' into feature/11-albidaya-wa-nihaya-vol-2

segullshairbutt committed:
feature/#11-albidaya-wa-nihaya-vol-2

Adds the quranic font details in here.

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
Update chap_3.md

muhammadnuman383 committed:
Update chap_3.md

muhammadnuman383 committed:
update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
Update chap_1.md

muhammadnuman383 committed:
Update chap_2.md

muhammadnuman383 committed:
Update chap_2.md

muhammadnuman383 committed:
Update chap_3.md

muhammadnuman383 committed:
Update chap_4.md

muhammadnuman383 committed:
Update chap_5.md

muhammadnuman383 committed:
Update chap_6.md

muhammadnuman383 committed:
update

muhammadnuman383 committed:
Update chap_8.md

muhammadnuman383 committed:
Update chap_8.md

muhammadnuman383 committed:
Update chap_8.md

muhammadnuman383 committed:
Update chap_9.md

muhammadnuman383 committed:
update

muhammadnuman383 committed:
Update chap_8.md

segullshairbutt committed:
feature/#11-albidaya-wa-nihaya-vol-2

Merges the main branch.

muhammadnuman383 committed:
Update chap_1.md

muhammadnuman383 committed:
Update chap_1.md

muhammadnuman383 committed:
Update chap_1.md

muhammadnuman383 committed:
Update chap_2.md

muhammadnuman383 committed:
update

muhammadnuman383 committed:
Update chap_3.md

muhammadnuman383 committed:
Update chap_3.md

muhammadnuman383 committed:
Update chap_3.md

muhammadnuman383 committed:
Update chap_4.md

muhammadnuman383 committed:
UPDATE

muhammadnuman383 committed:
Update chap_6.md

muhammadnuman383 committed:
update

muhammadnuman383 committed:
Update chap_6.md

muhammadnuman383 committed:
Update chap_6.md

muhammadnuman383 committed:
update

muhammadnuman383 committed:
Update chap_7.md

muhammadnuman383 committed:
Update chap_7.md

muhammadnuman383 committed:
update

muhammadnuman383 committed:
Update chap_8.md

muhammadnuman383 committed:
Update chap_8.md

muhammadnuman383 committed:
update

muhammadnuman383 committed:
Update chap_2.md

muhammadnuman383 committed:
update

update

muhammadnuman383 committed:
update

update

muhammadnuman383 committed:
update

update

muhammadnuman383 committed:
update

update

muhammadnuman383 committed:
update

muhammadnuman383 committed:
Merge branch 'feature/11-albidaya-wa-nihaya-vol-2' of https://github.com/segullshairbutt/islamopedia-books into feature/11-albidaya-wa-nihaya-vol-2

muhammadnuman383 committed:
Update chap_1.md

muhammadnuman383 committed:
Update chap_1.md

muhammadnuman383 committed:
Update chap_1.md

muhammadnuman383 committed:
Update chap_2.md

muhammadnuman383 committed:
Update chap_2.md

muhammadnuman383 committed:
Update chap_3.md

muhammadnuman383 committed:
Update chap_4.md

muhammadnuman383 committed:
Update chap_5.md

muhammadnuman383 committed:
Update chap_6.md

muhammadnuman383 committed:
Update chap_7.md

muhammadnuman383 committed:
Update chap_8.md

muhammadnuman383 committed:
Update chap_8.md

muhammadnuman383 committed:
Update chap_8.md

muhammadnuman383 committed:
update

muhammadnuman383 committed:
Update chap_9.md

muhammadnuman383 committed:
Update chap_9.md

muhammadnuman383 committed:
Update chap_9.md

muhammadnuman383 committed:
Update chap_8.md

muhammadnuman383 committed:
Update chap_8.md

muhammadnuman383 committed:
Update chap_9.md

muhammadnuman383 committed:
Update chap_1.md

muhammadnuman383 committed:
Update chap_8.md

muhammadnuman383 committed:
Update chap_9.md

segullshairbutt committed:
feature/#11: Albidaya wa nihaya volume 2

Fixes some formatting problems.

segullshairbutt committed:
Merge branch 'main' into feature/11-albidaya-wa-nihaya-vol-2

segullshairbutt committed:
feature/#11: Albidaya wa nihaya volume 2

Fixes some formatting problems.

segullshairbutt committed:
feature/#11: Albidaya wa nihaya volume 2

Fixes some formatting problems
